{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Transcriptome of tea tissues  \n",
    "\n",
    "*Xin Mei*  \n",
    "\n",
    "Pipeline ref: [Zika-RNAseq-Pipeline](http://nbviewer.jupyter.org/github/maayanlab/Zika-RNAseq-Pipeline/blob/master/Zika.ipynb). \n",
    " \n",
    "Data source: *Camellia sinensis* (L.) O. Kuntze cv. Longjing 43: [ref1](https://doi.org/10.1016/j.plaphy.2015.11.014), [ref2](https://doi.org/10.1016/j.scienta.2015.06.005), [ref3](https://doi.org/10.1186/1471-2164-14-415). [SRP034436](https://www.ncbi.nlm.nih.gov/Traces/study/?acc=SRP034436). [ftp](ftp://ftp.ncbi.nlm.nih.gov/sra/sra-instant/reads/ByRun/sra/SRR/SRR105/SRR1051214/).  "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Download SRA data from 'Accession List' of SRP034436:\n",
    "! nohup ~/local/src/sratoolkit.2.9.2-centos_linux64/bin/prefetch --option-file SRR_Acc_List.txt &"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "总用量 28G\r\n",
      "-rw-rw-r--. 1 meixin meixin 1.9G 3月   7 16:35 SRR1051214.sra\r\n",
      "-rw-rw-r--. 1 meixin meixin 2.8G 3月   7 16:37 SRR1053623.sra\r\n",
      "-rw-rw-r--. 1 meixin meixin 3.0G 3月   7 16:39 SRR1054007.sra\r\n",
      "-rw-rw-r--. 1 meixin meixin 2.0G 3月   7 16:40 SRR1054086.sra\r\n",
      "-rw-rw-r--. 1 meixin meixin 2.5G 3月   7 16:42 SRR1054152.sra\r\n",
      "-rw-rw-r--. 1 meixin meixin 2.0G 3月   7 16:44 SRR1055108.sra\r\n",
      "-rw-rw-r--. 1 meixin meixin 2.1G 3月   7 16:45 SRR1055109.sra\r\n",
      "-rw-rw-r--. 1 meixin meixin 1.5G 3月   7 16:46 SRR1055110.sra\r\n",
      "-rw-rw-r--. 1 meixin meixin 2.1G 3月   7 16:48 SRR1055182.sra\r\n",
      "-rw-rw-r--. 1 meixin meixin 2.1G 3月   7 16:50 SRR1055932.sra\r\n",
      "-rw-rw-r--. 1 meixin meixin 1.5G 3月   7 16:51 SRR1055933.sra\r\n",
      "-rw-rw-r--. 1 meixin meixin 2.1G 3月   7 16:53 SRR1055934.sra\r\n",
      "-rw-rw-r--. 1 meixin meixin 2.5G 3月   7 16:55 SRR1055944.sra\r\n"
     ]
    }
   ],
   "source": [
    "# Check the downloaded files:\n",
    "! ls -lh ~/ncbi/public/sra/"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Characterization of glutamate decarboxylases and calmodulin from tea (Camellia sinensis) leaves involved in GABA generation  "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "# transform 'General Feature Format' to 'Gene Transfer Format':\n",
    "! cp $GENOMEDIR/Annotation/sczGene.gff $GENOMEDIR/Annotation/sczGene.gtf\n",
    "! sed -i 's/mRNA/exon/' $GENOMEDIR/Annotation/sczGene.gtf\n",
    "! sed -i 's/ID=/gene_id \"/' $GENOMEDIR/Annotation/sczGene.gtf\n",
    "! sed -i 's/Parent=/transcript_id \"/' $GENOMEDIR/Annotation/sczGene.gtf\n",
    "! sed -i 's/gene_id\\(.*\\)/gene_id\\1transcript_id\\1/' $GENOMEDIR/Annotation/sczGene.gtf\n",
    "! sed -i 's/;/\"; /' $GENOMEDIR/Annotation/sczGene.gtf "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Number of CPUs to be used: 16\n",
      "GENOME=/home/meixin/genomes\n",
      "WORKDIR=/home/meixin/gadProperty/data\n",
      "Mar 17 17:03:31 ..... started STAR run\n",
      "Mar 17 17:03:31 ... starting to generate Genome files\n",
      "Mar 17 17:05:35 ... starting to sort Suffix Array. This may take a long time...\n",
      "Mar 17 17:06:21 ... sorting Suffix Array chunks and saving them to disk...\n",
      "Mar 17 19:02:21 ... loading chunks from disk, packing SA...\n",
      "Mar 17 19:08:43 ... finished generating suffix array\n",
      "Mar 17 19:08:43 ... generating Suffix Array index\n",
      "Mar 17 19:20:29 ... completed Suffix Array index\n",
      "Mar 17 19:20:29 ..... processing annotations GTF\n",
      "Mar 17 19:20:30 ... writing Genome to disk ...\n",
      "Mar 17 19:20:58 ... writing Suffix Array to disk ...\n",
      "Mar 17 19:23:12 ... writing SAindex to disk\n",
      "Mar 17 19:23:32 ..... finished successfully\n",
      "Dumping .sra files to .fastq.gz files\n",
      "/home/meixin/ncbi/public/sra/SRR1055182.sra is detected as paired-end sequencing reads\n",
      "/home/meixin/ncbi/public/sra/SRR1055934.sra is detected as paired-end sequencing reads\n",
      "/home/meixin/ncbi/public/sra/SRR1053623.sra is detected as paired-end sequencing reads\n",
      "/home/meixin/ncbi/public/sra/SRR1054007.sra is detected as paired-end sequencing reads\n",
      "/home/meixin/ncbi/public/sra/SRR1055110.sra is detected as paired-end sequencing reads\n",
      "/home/meixin/ncbi/public/sra/SRR1055944.sra is detected as paired-end sequencing reads\n",
      "/home/meixin/ncbi/public/sra/SRR1055932.sra is detected as paired-end sequencing reads\n",
      "/home/meixin/ncbi/public/sra/SRR1055108.sra is detected as paired-end sequencing reads\n",
      "/home/meixin/ncbi/public/sra/SRR1054152.sra is detected as paired-end sequencing reads\n",
      "/home/meixin/ncbi/public/sra/SRR1054086.sra is detected as paired-end sequencing reads\n",
      "/home/meixin/ncbi/public/sra/SRR1055933.sra is detected as paired-end sequencing reads\n",
      "/home/meixin/ncbi/public/sra/SRR1055109.sra is detected as paired-end sequencing reads\n",
      "/home/meixin/ncbi/public/sra/SRR1051214.sra is detected as paired-end sequencing reads\n",
      "Read 11105946 spots for /home/meixin/ncbi/public/sra/SRR1055933.sra\n",
      "Written 11105946 spots for /home/meixin/ncbi/public/sra/SRR1055933.sra\n",
      "Read 11179501 spots for /home/meixin/ncbi/public/sra/SRR1055110.sra\n",
      "Written 11179501 spots for /home/meixin/ncbi/public/sra/SRR1055110.sra\n",
      "Read 14673992 spots for /home/meixin/ncbi/public/sra/SRR1051214.sra\n",
      "Written 14673992 spots for /home/meixin/ncbi/public/sra/SRR1051214.sra\n",
      "Read 15668350 spots for /home/meixin/ncbi/public/sra/SRR1055108.sra\n",
      "Written 15668350 spots for /home/meixin/ncbi/public/sra/SRR1055108.sra\n",
      "Read 16549011 spots for /home/meixin/ncbi/public/sra/SRR1055109.sra\n",
      "Written 16549011 spots for /home/meixin/ncbi/public/sra/SRR1055109.sra\n",
      "Read 15781193 spots for /home/meixin/ncbi/public/sra/SRR1055934.sra\n",
      "Written 15781193 spots for /home/meixin/ncbi/public/sra/SRR1055934.sra\n",
      "Read 16418027 spots for /home/meixin/ncbi/public/sra/SRR1055932.sra\n",
      "Written 16418027 spots for /home/meixin/ncbi/public/sra/SRR1055932.sra\n",
      "Read 16765245 spots for /home/meixin/ncbi/public/sra/SRR1055182.sra\n",
      "Written 16765245 spots for /home/meixin/ncbi/public/sra/SRR1055182.sra\n",
      "Read 15799091 spots for /home/meixin/ncbi/public/sra/SRR1054086.sra\n",
      "Written 15799091 spots for /home/meixin/ncbi/public/sra/SRR1054086.sra\n",
      "Read 19816534 spots for /home/meixin/ncbi/public/sra/SRR1055944.sra\n",
      "Written 19816534 spots for /home/meixin/ncbi/public/sra/SRR1055944.sra\n",
      "Read 20796054 spots for /home/meixin/ncbi/public/sra/SRR1054152.sra\n",
      "Written 20796054 spots for /home/meixin/ncbi/public/sra/SRR1054152.sra\n",
      "Read 21132603 spots for /home/meixin/ncbi/public/sra/SRR1053623.sra\n",
      "Written 21132603 spots for /home/meixin/ncbi/public/sra/SRR1053623.sra\n",
      "Read 22986435 spots for /home/meixin/ncbi/public/sra/SRR1054007.sra\n",
      "Written 22986435 spots for /home/meixin/ncbi/public/sra/SRR1054007.sra\n",
      "Started to align reads to the genome and assemble transcriptome\n",
      "SRR1051214\n",
      "Performing FastQC for SRR1051214\n",
      "Started analysis of SRR1051214_1.fastq.gz\n",
      "Approx 5% complete for SRR1051214_1.fastq.gz\n",
      "Approx 10% complete for SRR1051214_1.fastq.gz\n",
      "Approx 15% complete for SRR1051214_1.fastq.gz\n",
      "Approx 20% complete for SRR1051214_1.fastq.gz\n",
      "Approx 25% complete for SRR1051214_1.fastq.gz\n",
      "Approx 30% complete for SRR1051214_1.fastq.gz\n",
      "Approx 35% complete for SRR1051214_1.fastq.gz\n",
      "Approx 40% complete for SRR1051214_1.fastq.gz\n",
      "Approx 45% complete for SRR1051214_1.fastq.gz\n",
      "Approx 50% complete for SRR1051214_1.fastq.gz\n",
      "Approx 55% complete for SRR1051214_1.fastq.gz\n",
      "Approx 60% complete for SRR1051214_1.fastq.gz\n",
      "Approx 65% complete for SRR1051214_1.fastq.gz\n",
      "Approx 70% complete for SRR1051214_1.fastq.gz\n",
      "Approx 75% complete for SRR1051214_1.fastq.gz\n",
      "Approx 80% complete for SRR1051214_1.fastq.gz\n",
      "Approx 85% complete for SRR1051214_1.fastq.gz\n",
      "Approx 90% complete for SRR1051214_1.fastq.gz\n",
      "Approx 95% complete for SRR1051214_1.fastq.gz\n",
      "Analysis complete for SRR1051214_1.fastq.gz\n",
      "Started analysis of SRR1051214_2.fastq.gz\n",
      "Approx 5% complete for SRR1051214_2.fastq.gz\n",
      "Approx 10% complete for SRR1051214_2.fastq.gz\n",
      "Approx 15% complete for SRR1051214_2.fastq.gz\n",
      "Approx 20% complete for SRR1051214_2.fastq.gz\n",
      "Approx 25% complete for SRR1051214_2.fastq.gz\n",
      "Approx 30% complete for SRR1051214_2.fastq.gz\n",
      "Approx 35% complete for SRR1051214_2.fastq.gz\n",
      "Approx 40% complete for SRR1051214_2.fastq.gz\n",
      "Approx 45% complete for SRR1051214_2.fastq.gz\n",
      "Approx 50% complete for SRR1051214_2.fastq.gz\n",
      "Approx 55% complete for SRR1051214_2.fastq.gz\n",
      "Approx 60% complete for SRR1051214_2.fastq.gz\n",
      "Approx 65% complete for SRR1051214_2.fastq.gz\n",
      "Approx 70% complete for SRR1051214_2.fastq.gz\n",
      "Approx 75% complete for SRR1051214_2.fastq.gz\n",
      "Approx 80% complete for SRR1051214_2.fastq.gz\n",
      "Approx 85% complete for SRR1051214_2.fastq.gz\n",
      "Approx 90% complete for SRR1051214_2.fastq.gz\n",
      "Approx 95% complete for SRR1051214_2.fastq.gz\n",
      "Analysis complete for SRR1051214_2.fastq.gz\n",
      "Aligning reads from SRR1051214 to the reference genome\n",
      "Mar 17 20:36:34 ..... started STAR run\n",
      "Mar 17 20:36:34 ..... loading genome\n",
      "Mar 17 20:39:30 ..... processing annotations GTF\n",
      "Mar 17 20:39:32 ..... started mapping\n",
      "Mar 17 21:28:28 ..... finished successfully\n",
      "\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m    _____ _    _ ____  _____  ______          _____  \n",
      "       \u001b[44;37m =====      \u001b[0m\u001b[36m   / ____| |  | |  _ \\|  __ \\|  ____|   /\\   |  __ \\ \n",
      "       \u001b[44;37m   =====    \u001b[0m\u001b[36m  | (___ | |  | | |_) | |__) | |__     /  \\  | |  | |\n",
      "       \u001b[44;37m     ====   \u001b[0m\u001b[36m   \\___ \\| |  | |  _ <|  _  /|  __|   / /\\ \\ | |  | |\n",
      "       \u001b[44;37m       ==== \u001b[0m\u001b[36m   ____) | |__| | |_) | | \\ \\| |____ / ____ \\| |__| |\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m  |_____/ \\____/|____/|_|  \\_\\______/_/    \\_\\_____/\u001b[0m\n",
      "\t  v1.6.3\n",
      "\n",
      "//==========================\u001b[36m featureCounts setting \u001b[0m===========================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Input files : \u001b[36m1 BAM file  \u001b[0m \u001b[0m                                    ||\n",
      "||                           \u001b[32mP\u001b[36m SRR1051214Aligned.out.bam\u001b[0m \u001b[0m                     ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Output file : \u001b[36mSRR1051214.count.txt\u001b[0m \u001b[0m                            ||\n",
      "||                 Summary : \u001b[36mSRR1051214.count.txt.summary\u001b[0m \u001b[0m                    ||\n",
      "||              Annotation : \u001b[36msczGene.gtf (GTF)\u001b[0m \u001b[0m                               ||\n",
      "||      Dir for temp files : \u001b[36m../featureCount_output\u001b[0m \u001b[0m                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||                 Threads : \u001b[36m16\u001b[0m \u001b[0m                                              ||\n",
      "||                   Level : \u001b[36mmeta-feature level\u001b[0m \u001b[0m                              ||\n",
      "||              Paired-end : \u001b[36mno\u001b[0m \u001b[0m                                              ||\n",
      "||      Multimapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "|| Multi-overlapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "||   Min overlapping bases : \u001b[36m1\u001b[0m \u001b[0m                                               ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\=====================\u001b[36m http://subread.sourceforge.net/ \u001b[0m======================//\n",
      "\n",
      "//=================================\u001b[36m Running \u001b[0m==================================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Load annotation file sczGene.gtf \u001b[0m... \u001b[0m                                      ||\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "||    Features : \u001b[36m33932\u001b[0m \u001b[0m                                                       ||\n",
      "||    Meta-features : \u001b[36m33932\u001b[0m \u001b[0m                                                  ||\n",
      "||    Chromosomes/contigs : \u001b[36m3116\u001b[0m \u001b[0m                                             ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Process BAM file SRR1051214Aligned.out.bam... \u001b[0m                             ||\n",
      "||    Paired-end reads are included. \u001b[0m                                         ||\n",
      "||    Assign alignments to features... \u001b[0m                                       ||\n",
      "||    Total alignments : \u001b[36m53872878\u001b[0m \u001b[0m                                            ||\n",
      "||    Successfully assigned alignments : \u001b[36m10158549 (18.9%)\u001b[0m \u001b[0m                    ||\n",
      "||    Running time : \u001b[36m0.64 minutes\u001b[0m \u001b[0m                                            ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Summary of counting results can be found in file \"../featureCount_output/ \u001b[0m ||\n",
      "|| SRR1051214.count.txt.summary\" \u001b[0m                                             ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\=====================\u001b[36m http://subread.sourceforge.net/ \u001b[0m======================//\n",
      "\n",
      "SRR1053623\n",
      "Performing FastQC for SRR1053623\n",
      "Started analysis of SRR1053623_1.fastq.gz\n",
      "Approx 5% complete for SRR1053623_1.fastq.gz\n",
      "Approx 10% complete for SRR1053623_1.fastq.gz\n",
      "Approx 15% complete for SRR1053623_1.fastq.gz\n",
      "Approx 20% complete for SRR1053623_1.fastq.gz\n",
      "Approx 25% complete for SRR1053623_1.fastq.gz\n",
      "Approx 30% complete for SRR1053623_1.fastq.gz\n",
      "Approx 35% complete for SRR1053623_1.fastq.gz\n",
      "Approx 40% complete for SRR1053623_1.fastq.gz\n",
      "Approx 45% complete for SRR1053623_1.fastq.gz\n",
      "Approx 50% complete for SRR1053623_1.fastq.gz\n",
      "Approx 55% complete for SRR1053623_1.fastq.gz\n",
      "Approx 60% complete for SRR1053623_1.fastq.gz\n",
      "Approx 65% complete for SRR1053623_1.fastq.gz\n",
      "Approx 70% complete for SRR1053623_1.fastq.gz\n",
      "Approx 75% complete for SRR1053623_1.fastq.gz\n",
      "Approx 80% complete for SRR1053623_1.fastq.gz\n",
      "Approx 85% complete for SRR1053623_1.fastq.gz\n",
      "Approx 90% complete for SRR1053623_1.fastq.gz\n",
      "Approx 95% complete for SRR1053623_1.fastq.gz\n",
      "Analysis complete for SRR1053623_1.fastq.gz\n",
      "Started analysis of SRR1053623_2.fastq.gz\n",
      "Approx 5% complete for SRR1053623_2.fastq.gz\n",
      "Approx 10% complete for SRR1053623_2.fastq.gz\n",
      "Approx 15% complete for SRR1053623_2.fastq.gz\n",
      "Approx 20% complete for SRR1053623_2.fastq.gz\n",
      "Approx 25% complete for SRR1053623_2.fastq.gz\n",
      "Approx 30% complete for SRR1053623_2.fastq.gz\n",
      "Approx 35% complete for SRR1053623_2.fastq.gz\n",
      "Approx 40% complete for SRR1053623_2.fastq.gz\n",
      "Approx 45% complete for SRR1053623_2.fastq.gz\n",
      "Approx 50% complete for SRR1053623_2.fastq.gz\n",
      "Approx 55% complete for SRR1053623_2.fastq.gz\n",
      "Approx 60% complete for SRR1053623_2.fastq.gz\n",
      "Approx 65% complete for SRR1053623_2.fastq.gz\n",
      "Approx 70% complete for SRR1053623_2.fastq.gz\n",
      "Approx 75% complete for SRR1053623_2.fastq.gz\n",
      "Approx 80% complete for SRR1053623_2.fastq.gz\n",
      "Approx 85% complete for SRR1053623_2.fastq.gz\n",
      "Approx 90% complete for SRR1053623_2.fastq.gz\n",
      "Approx 95% complete for SRR1053623_2.fastq.gz\n",
      "Analysis complete for SRR1053623_2.fastq.gz\n",
      "Aligning reads from SRR1053623 to the reference genome\n",
      "Mar 17 21:41:24 ..... started STAR run\n",
      "Mar 17 21:41:24 ..... loading genome\n",
      "Mar 17 21:44:09 ..... processing annotations GTF\n",
      "Mar 17 21:44:10 ..... started mapping\n",
      "Mar 17 22:14:09 ..... finished successfully\n",
      "\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m    _____ _    _ ____  _____  ______          _____  \n",
      "       \u001b[44;37m =====      \u001b[0m\u001b[36m   / ____| |  | |  _ \\|  __ \\|  ____|   /\\   |  __ \\ \n",
      "       \u001b[44;37m   =====    \u001b[0m\u001b[36m  | (___ | |  | | |_) | |__) | |__     /  \\  | |  | |\n",
      "       \u001b[44;37m     ====   \u001b[0m\u001b[36m   \\___ \\| |  | |  _ <|  _  /|  __|   / /\\ \\ | |  | |\n",
      "       \u001b[44;37m       ==== \u001b[0m\u001b[36m   ____) | |__| | |_) | | \\ \\| |____ / ____ \\| |__| |\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m  |_____/ \\____/|____/|_|  \\_\\______/_/    \\_\\_____/\u001b[0m\n",
      "\t  v1.6.3\n",
      "\n",
      "//==========================\u001b[36m featureCounts setting \u001b[0m===========================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Input files : \u001b[36m1 BAM file  \u001b[0m \u001b[0m                                    ||\n",
      "||                           \u001b[32mP\u001b[36m SRR1053623Aligned.out.bam\u001b[0m \u001b[0m                     ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Output file : \u001b[36mSRR1053623.count.txt\u001b[0m \u001b[0m                            ||\n",
      "||                 Summary : \u001b[36mSRR1053623.count.txt.summary\u001b[0m \u001b[0m                    ||\n",
      "||              Annotation : \u001b[36msczGene.gtf (GTF)\u001b[0m \u001b[0m                               ||\n",
      "||      Dir for temp files : \u001b[36m../featureCount_output\u001b[0m \u001b[0m                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||                 Threads : \u001b[36m16\u001b[0m \u001b[0m                                              ||\n",
      "||                   Level : \u001b[36mmeta-feature level\u001b[0m \u001b[0m                              ||\n",
      "||              Paired-end : \u001b[36mno\u001b[0m \u001b[0m                                              ||\n",
      "||      Multimapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "|| Multi-overlapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "||   Min overlapping bases : \u001b[36m1\u001b[0m \u001b[0m                                               ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\=====================\u001b[36m http://subread.sourceforge.net/ \u001b[0m======================//\n",
      "\n",
      "//=================================\u001b[36m Running \u001b[0m==================================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Load annotation file sczGene.gtf \u001b[0m... \u001b[0m                                      ||\n",
      "||    Features : \u001b[36m33932\u001b[0m \u001b[0m                                                       ||\n",
      "||    Meta-features : \u001b[36m33932\u001b[0m \u001b[0m                                                  ||\n",
      "||    Chromosomes/contigs : \u001b[36m3116\u001b[0m \u001b[0m                                             ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Process BAM file SRR1053623Aligned.out.bam... \u001b[0m                             ||\n",
      "||    Paired-end reads are included. \u001b[0m                                         ||\n",
      "||    Assign alignments to features... \u001b[0m                                       ||\n",
      "||    Total alignments : \u001b[36m44315052\u001b[0m \u001b[0m                                            ||\n",
      "||    Successfully assigned alignments : \u001b[36m27766700 (62.7%)\u001b[0m \u001b[0m                    ||\n",
      "||    Running time : \u001b[36m0.86 minutes\u001b[0m \u001b[0m                                            ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Summary of counting results can be found in file \"../featureCount_output/ \u001b[0m ||\n",
      "|| SRR1053623.count.txt.summary\" \u001b[0m                                             ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\=====================\u001b[36m http://subread.sourceforge.net/ \u001b[0m======================//\n",
      "\n",
      "SRR1054007\n",
      "Performing FastQC for SRR1054007\n",
      "Started analysis of SRR1054007_1.fastq.gz\n",
      "Approx 5% complete for SRR1054007_1.fastq.gz\n",
      "Approx 10% complete for SRR1054007_1.fastq.gz\n",
      "Approx 15% complete for SRR1054007_1.fastq.gz\n",
      "Approx 20% complete for SRR1054007_1.fastq.gz\n",
      "Approx 25% complete for SRR1054007_1.fastq.gz\n",
      "Approx 30% complete for SRR1054007_1.fastq.gz\n",
      "Approx 35% complete for SRR1054007_1.fastq.gz\n",
      "Approx 40% complete for SRR1054007_1.fastq.gz\n",
      "Approx 45% complete for SRR1054007_1.fastq.gz\n",
      "Approx 50% complete for SRR1054007_1.fastq.gz\n",
      "Approx 55% complete for SRR1054007_1.fastq.gz\n",
      "Approx 60% complete for SRR1054007_1.fastq.gz\n",
      "Approx 65% complete for SRR1054007_1.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 70% complete for SRR1054007_1.fastq.gz\n",
      "Approx 75% complete for SRR1054007_1.fastq.gz\n",
      "Approx 80% complete for SRR1054007_1.fastq.gz\n",
      "Approx 85% complete for SRR1054007_1.fastq.gz\n",
      "Approx 90% complete for SRR1054007_1.fastq.gz\n",
      "Approx 95% complete for SRR1054007_1.fastq.gz\n",
      "Analysis complete for SRR1054007_1.fastq.gz\n",
      "Started analysis of SRR1054007_2.fastq.gz\n",
      "Approx 5% complete for SRR1054007_2.fastq.gz\n",
      "Approx 10% complete for SRR1054007_2.fastq.gz\n",
      "Approx 15% complete for SRR1054007_2.fastq.gz\n",
      "Approx 20% complete for SRR1054007_2.fastq.gz\n",
      "Approx 25% complete for SRR1054007_2.fastq.gz\n",
      "Approx 30% complete for SRR1054007_2.fastq.gz\n",
      "Approx 35% complete for SRR1054007_2.fastq.gz\n",
      "Approx 40% complete for SRR1054007_2.fastq.gz\n",
      "Approx 45% complete for SRR1054007_2.fastq.gz\n",
      "Approx 50% complete for SRR1054007_2.fastq.gz\n",
      "Approx 55% complete for SRR1054007_2.fastq.gz\n",
      "Approx 60% complete for SRR1054007_2.fastq.gz\n",
      "Approx 65% complete for SRR1054007_2.fastq.gz\n",
      "Approx 70% complete for SRR1054007_2.fastq.gz\n",
      "Approx 75% complete for SRR1054007_2.fastq.gz\n",
      "Approx 80% complete for SRR1054007_2.fastq.gz\n",
      "Approx 85% complete for SRR1054007_2.fastq.gz\n",
      "Approx 90% complete for SRR1054007_2.fastq.gz\n",
      "Approx 95% complete for SRR1054007_2.fastq.gz\n",
      "Analysis complete for SRR1054007_2.fastq.gz\n",
      "Aligning reads from SRR1054007 to the reference genome\n",
      "Mar 17 22:27:33 ..... started STAR run\n",
      "Mar 17 22:27:33 ..... loading genome\n",
      "Mar 17 22:30:20 ..... processing annotations GTF\n",
      "Mar 17 22:30:22 ..... started mapping\n",
      "Mar 17 23:00:57 ..... finished successfully\n",
      "\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m    _____ _    _ ____  _____  ______          _____  \n",
      "       \u001b[44;37m =====      \u001b[0m\u001b[36m   / ____| |  | |  _ \\|  __ \\|  ____|   /\\   |  __ \\ \n",
      "       \u001b[44;37m   =====    \u001b[0m\u001b[36m  | (___ | |  | | |_) | |__) | |__     /  \\  | |  | |\n",
      "       \u001b[44;37m     ====   \u001b[0m\u001b[36m   \\___ \\| |  | |  _ <|  _  /|  __|   / /\\ \\ | |  | |\n",
      "       \u001b[44;37m       ==== \u001b[0m\u001b[36m   ____) | |__| | |_) | | \\ \\| |____ / ____ \\| |__| |\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m  |_____/ \\____/|____/|_|  \\_\\______/_/    \\_\\_____/\u001b[0m\n",
      "\t  v1.6.3\n",
      "\n",
      "//==========================\u001b[36m featureCounts setting \u001b[0m===========================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Input files : \u001b[36m1 BAM file  \u001b[0m \u001b[0m                                    ||\n",
      "||                           \u001b[32mP\u001b[36m SRR1054007Aligned.out.bam\u001b[0m \u001b[0m                     ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Output file : \u001b[36mSRR1054007.count.txt\u001b[0m \u001b[0m                            ||\n",
      "||                 Summary : \u001b[36mSRR1054007.count.txt.summary\u001b[0m \u001b[0m                    ||\n",
      "||              Annotation : \u001b[36msczGene.gtf (GTF)\u001b[0m \u001b[0m                               ||\n",
      "||      Dir for temp files : \u001b[36m../featureCount_output\u001b[0m \u001b[0m                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||                 Threads : \u001b[36m16\u001b[0m \u001b[0m                                              ||\n",
      "||                   Level : \u001b[36mmeta-feature level\u001b[0m \u001b[0m                              ||\n",
      "||              Paired-end : \u001b[36mno\u001b[0m \u001b[0m                                              ||\n",
      "||      Multimapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "|| Multi-overlapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "||   Min overlapping bases : \u001b[36m1\u001b[0m \u001b[0m                                               ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\=====================\u001b[36m http://subread.sourceforge.net/ \u001b[0m======================//\n",
      "\n",
      "//=================================\u001b[36m Running \u001b[0m==================================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Load annotation file sczGene.gtf \u001b[0m... \u001b[0m                                      ||\n",
      "||    Features : \u001b[36m33932\u001b[0m \u001b[0m                                                       ||\n",
      "||    Meta-features : \u001b[36m33932\u001b[0m \u001b[0m                                                  ||\n",
      "||    Chromosomes/contigs : \u001b[36m3116\u001b[0m \u001b[0m                                             ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Process BAM file SRR1054007Aligned.out.bam... \u001b[0m                             ||\n",
      "||    Paired-end reads are included. \u001b[0m                                         ||\n",
      "||    Assign alignments to features... \u001b[0m                                       ||\n",
      "||    Total alignments : \u001b[36m47283124\u001b[0m \u001b[0m                                            ||\n",
      "||    Successfully assigned alignments : \u001b[36m29976087 (63.4%)\u001b[0m \u001b[0m                    ||\n",
      "||    Running time : \u001b[36m0.95 minutes\u001b[0m \u001b[0m                                            ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Summary of counting results can be found in file \"../featureCount_output/ \u001b[0m ||\n",
      "|| SRR1054007.count.txt.summary\" \u001b[0m                                             ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\=====================\u001b[36m http://subread.sourceforge.net/ \u001b[0m======================//\n",
      "\n",
      "SRR1054086\n",
      "Performing FastQC for SRR1054086\n",
      "Started analysis of SRR1054086_1.fastq.gz\n",
      "Approx 5% complete for SRR1054086_1.fastq.gz\n",
      "Approx 10% complete for SRR1054086_1.fastq.gz\n",
      "Approx 15% complete for SRR1054086_1.fastq.gz\n",
      "Approx 20% complete for SRR1054086_1.fastq.gz\n",
      "Approx 25% complete for SRR1054086_1.fastq.gz\n",
      "Approx 30% complete for SRR1054086_1.fastq.gz\n",
      "Approx 35% complete for SRR1054086_1.fastq.gz\n",
      "Approx 40% complete for SRR1054086_1.fastq.gz\n",
      "Approx 45% complete for SRR1054086_1.fastq.gz\n",
      "Approx 50% complete for SRR1054086_1.fastq.gz\n",
      "Approx 55% complete for SRR1054086_1.fastq.gz\n",
      "Approx 60% complete for SRR1054086_1.fastq.gz\n",
      "Approx 65% complete for SRR1054086_1.fastq.gz\n",
      "Approx 70% complete for SRR1054086_1.fastq.gz\n",
      "Approx 75% complete for SRR1054086_1.fastq.gz\n",
      "Approx 80% complete for SRR1054086_1.fastq.gz\n",
      "Approx 85% complete for SRR1054086_1.fastq.gz\n",
      "Approx 90% complete for SRR1054086_1.fastq.gz\n",
      "Approx 95% complete for SRR1054086_1.fastq.gz\n",
      "Analysis complete for SRR1054086_1.fastq.gz\n",
      "Started analysis of SRR1054086_2.fastq.gz\n",
      "Approx 5% complete for SRR1054086_2.fastq.gz\n",
      "Approx 10% complete for SRR1054086_2.fastq.gz\n",
      "Approx 15% complete for SRR1054086_2.fastq.gz\n",
      "Approx 20% complete for SRR1054086_2.fastq.gz\n",
      "Approx 25% complete for SRR1054086_2.fastq.gz\n",
      "Approx 30% complete for SRR1054086_2.fastq.gz\n",
      "Approx 35% complete for SRR1054086_2.fastq.gz\n",
      "Approx 40% complete for SRR1054086_2.fastq.gz\n",
      "Approx 45% complete for SRR1054086_2.fastq.gz\n",
      "Approx 50% complete for SRR1054086_2.fastq.gz\n",
      "Approx 55% complete for SRR1054086_2.fastq.gz\n",
      "Approx 60% complete for SRR1054086_2.fastq.gz\n",
      "Approx 65% complete for SRR1054086_2.fastq.gz\n",
      "Approx 70% complete for SRR1054086_2.fastq.gz\n",
      "Approx 75% complete for SRR1054086_2.fastq.gz\n",
      "Approx 80% complete for SRR1054086_2.fastq.gz\n",
      "Approx 85% complete for SRR1054086_2.fastq.gz\n",
      "Approx 90% complete for SRR1054086_2.fastq.gz\n",
      "Approx 95% complete for SRR1054086_2.fastq.gz\n",
      "Analysis complete for SRR1054086_2.fastq.gz\n",
      "Aligning reads from SRR1054086 to the reference genome\n",
      "Mar 17 23:11:11 ..... started STAR run\n",
      "Mar 17 23:11:11 ..... loading genome\n",
      "Mar 17 23:14:01 ..... processing annotations GTF\n",
      "Mar 17 23:14:03 ..... started mapping\n",
      "Mar 17 23:35:05 ..... finished successfully\n",
      "\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m    _____ _    _ ____  _____  ______          _____  \n",
      "       \u001b[44;37m =====      \u001b[0m\u001b[36m   / ____| |  | |  _ \\|  __ \\|  ____|   /\\   |  __ \\ \n",
      "       \u001b[44;37m   =====    \u001b[0m\u001b[36m  | (___ | |  | | |_) | |__) | |__     /  \\  | |  | |\n",
      "       \u001b[44;37m     ====   \u001b[0m\u001b[36m   \\___ \\| |  | |  _ <|  _  /|  __|   / /\\ \\ | |  | |\n",
      "       \u001b[44;37m       ==== \u001b[0m\u001b[36m   ____) | |__| | |_) | | \\ \\| |____ / ____ \\| |__| |\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m  |_____/ \\____/|____/|_|  \\_\\______/_/    \\_\\_____/\u001b[0m\n",
      "\t  v1.6.3\n",
      "\n",
      "//==========================\u001b[36m featureCounts setting \u001b[0m===========================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Input files : \u001b[36m1 BAM file  \u001b[0m \u001b[0m                                    ||\n",
      "||                           \u001b[32mP\u001b[36m SRR1054086Aligned.out.bam\u001b[0m \u001b[0m                     ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Output file : \u001b[36mSRR1054086.count.txt\u001b[0m \u001b[0m                            ||\n",
      "||                 Summary : \u001b[36mSRR1054086.count.txt.summary\u001b[0m \u001b[0m                    ||\n",
      "||              Annotation : \u001b[36msczGene.gtf (GTF)\u001b[0m \u001b[0m                               ||\n",
      "||      Dir for temp files : \u001b[36m../featureCount_output\u001b[0m \u001b[0m                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||                 Threads : \u001b[36m16\u001b[0m \u001b[0m                                              ||\n",
      "||                   Level : \u001b[36mmeta-feature level\u001b[0m \u001b[0m                              ||\n",
      "||              Paired-end : \u001b[36mno\u001b[0m \u001b[0m                                              ||\n",
      "||      Multimapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "|| Multi-overlapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "||   Min overlapping bases : \u001b[36m1\u001b[0m \u001b[0m                                               ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\=====================\u001b[36m http://subread.sourceforge.net/ \u001b[0m======================//\n",
      "\n",
      "//=================================\u001b[36m Running \u001b[0m==================================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Load annotation file sczGene.gtf \u001b[0m... \u001b[0m                                      ||\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "||    Features : \u001b[36m33932\u001b[0m \u001b[0m                                                       ||\n",
      "||    Meta-features : \u001b[36m33932\u001b[0m \u001b[0m                                                  ||\n",
      "||    Chromosomes/contigs : \u001b[36m3116\u001b[0m \u001b[0m                                             ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Process BAM file SRR1054086Aligned.out.bam... \u001b[0m                             ||\n",
      "||    Paired-end reads are included. \u001b[0m                                         ||\n",
      "||    Assign alignments to features... \u001b[0m                                       ||\n",
      "||    Total alignments : \u001b[36m31684836\u001b[0m \u001b[0m                                            ||\n",
      "||    Successfully assigned alignments : \u001b[36m22092695 (69.7%)\u001b[0m \u001b[0m                    ||\n",
      "||    Running time : \u001b[36m0.35 minutes\u001b[0m \u001b[0m                                            ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Summary of counting results can be found in file \"../featureCount_output/ \u001b[0m ||\n",
      "|| SRR1054086.count.txt.summary\" \u001b[0m                                             ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\=====================\u001b[36m http://subread.sourceforge.net/ \u001b[0m======================//\n",
      "\n",
      "SRR1054152\n",
      "Performing FastQC for SRR1054152\n",
      "Started analysis of SRR1054152_1.fastq.gz\n",
      "Approx 5% complete for SRR1054152_1.fastq.gz\n",
      "Approx 10% complete for SRR1054152_1.fastq.gz\n",
      "Approx 15% complete for SRR1054152_1.fastq.gz\n",
      "Approx 20% complete for SRR1054152_1.fastq.gz\n",
      "Approx 25% complete for SRR1054152_1.fastq.gz\n",
      "Approx 30% complete for SRR1054152_1.fastq.gz\n",
      "Approx 35% complete for SRR1054152_1.fastq.gz\n",
      "Approx 40% complete for SRR1054152_1.fastq.gz\n",
      "Approx 45% complete for SRR1054152_1.fastq.gz\n",
      "Approx 50% complete for SRR1054152_1.fastq.gz\n",
      "Approx 55% complete for SRR1054152_1.fastq.gz\n",
      "Approx 60% complete for SRR1054152_1.fastq.gz\n",
      "Approx 65% complete for SRR1054152_1.fastq.gz\n",
      "Approx 70% complete for SRR1054152_1.fastq.gz\n",
      "Approx 75% complete for SRR1054152_1.fastq.gz\n",
      "Approx 80% complete for SRR1054152_1.fastq.gz\n",
      "Approx 85% complete for SRR1054152_1.fastq.gz\n",
      "Approx 90% complete for SRR1054152_1.fastq.gz\n",
      "Approx 95% complete for SRR1054152_1.fastq.gz\n",
      "Analysis complete for SRR1054152_1.fastq.gz\n",
      "Started analysis of SRR1054152_2.fastq.gz\n",
      "Approx 5% complete for SRR1054152_2.fastq.gz\n",
      "Approx 10% complete for SRR1054152_2.fastq.gz\n",
      "Approx 15% complete for SRR1054152_2.fastq.gz\n",
      "Approx 20% complete for SRR1054152_2.fastq.gz\n",
      "Approx 25% complete for SRR1054152_2.fastq.gz\n",
      "Approx 30% complete for SRR1054152_2.fastq.gz\n",
      "Approx 35% complete for SRR1054152_2.fastq.gz\n",
      "Approx 40% complete for SRR1054152_2.fastq.gz\n",
      "Approx 45% complete for SRR1054152_2.fastq.gz\n",
      "Approx 50% complete for SRR1054152_2.fastq.gz\n",
      "Approx 55% complete for SRR1054152_2.fastq.gz\n",
      "Approx 60% complete for SRR1054152_2.fastq.gz\n",
      "Approx 65% complete for SRR1054152_2.fastq.gz\n",
      "Approx 70% complete for SRR1054152_2.fastq.gz\n",
      "Approx 75% complete for SRR1054152_2.fastq.gz\n",
      "Approx 80% complete for SRR1054152_2.fastq.gz\n",
      "Approx 85% complete for SRR1054152_2.fastq.gz\n",
      "Approx 90% complete for SRR1054152_2.fastq.gz\n",
      "Approx 95% complete for SRR1054152_2.fastq.gz\n",
      "Analysis complete for SRR1054152_2.fastq.gz\n",
      "Aligning reads from SRR1054152 to the reference genome\n",
      "Mar 17 23:47:21 ..... started STAR run\n",
      "Mar 17 23:47:21 ..... loading genome\n",
      "Mar 17 23:50:12 ..... processing annotations GTF\n",
      "Mar 17 23:50:13 ..... started mapping\n",
      "Mar 18 00:24:38 ..... finished successfully\n",
      "\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m    _____ _    _ ____  _____  ______          _____  \n",
      "       \u001b[44;37m =====      \u001b[0m\u001b[36m   / ____| |  | |  _ \\|  __ \\|  ____|   /\\   |  __ \\ \n",
      "       \u001b[44;37m   =====    \u001b[0m\u001b[36m  | (___ | |  | | |_) | |__) | |__     /  \\  | |  | |\n",
      "       \u001b[44;37m     ====   \u001b[0m\u001b[36m   \\___ \\| |  | |  _ <|  _  /|  __|   / /\\ \\ | |  | |\n",
      "       \u001b[44;37m       ==== \u001b[0m\u001b[36m   ____) | |__| | |_) | | \\ \\| |____ / ____ \\| |__| |\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m  |_____/ \\____/|____/|_|  \\_\\______/_/    \\_\\_____/\u001b[0m\n",
      "\t  v1.6.3\n",
      "\n",
      "//==========================\u001b[36m featureCounts setting \u001b[0m===========================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Input files : \u001b[36m1 BAM file  \u001b[0m \u001b[0m                                    ||\n",
      "||                           \u001b[32mP\u001b[36m SRR1054152Aligned.out.bam\u001b[0m \u001b[0m                     ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Output file : \u001b[36mSRR1054152.count.txt\u001b[0m \u001b[0m                            ||\n",
      "||                 Summary : \u001b[36mSRR1054152.count.txt.summary\u001b[0m \u001b[0m                    ||\n",
      "||              Annotation : \u001b[36msczGene.gtf (GTF)\u001b[0m \u001b[0m                               ||\n",
      "||      Dir for temp files : \u001b[36m../featureCount_output\u001b[0m \u001b[0m                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||                 Threads : \u001b[36m16\u001b[0m \u001b[0m                                              ||\n",
      "||                   Level : \u001b[36mmeta-feature level\u001b[0m \u001b[0m                              ||\n",
      "||              Paired-end : \u001b[36mno\u001b[0m \u001b[0m                                              ||\n",
      "||      Multimapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "|| Multi-overlapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "||   Min overlapping bases : \u001b[36m1\u001b[0m \u001b[0m                                               ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\=====================\u001b[36m http://subread.sourceforge.net/ \u001b[0m======================//\n",
      "\n",
      "//=================================\u001b[36m Running \u001b[0m==================================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Load annotation file sczGene.gtf \u001b[0m... \u001b[0m                                      ||\n",
      "||    Features : \u001b[36m33932\u001b[0m \u001b[0m                                                       ||\n",
      "||    Meta-features : \u001b[36m33932\u001b[0m \u001b[0m                                                  ||\n",
      "||    Chromosomes/contigs : \u001b[36m3116\u001b[0m \u001b[0m                                             ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Process BAM file SRR1054152Aligned.out.bam... \u001b[0m                             ||\n",
      "||    Paired-end reads are included. \u001b[0m                                         ||\n",
      "||    Assign alignments to features... \u001b[0m                                       ||\n",
      "||    Total alignments : \u001b[36m32570640\u001b[0m \u001b[0m                                            ||\n",
      "||    Successfully assigned alignments : \u001b[36m21114732 (64.8%)\u001b[0m \u001b[0m                    ||\n",
      "||    Running time : \u001b[36m0.75 minutes\u001b[0m \u001b[0m                                            ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Summary of counting results can be found in file \"../featureCount_output/ \u001b[0m ||\n",
      "|| SRR1054152.count.txt.summary\" \u001b[0m                                             ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\=====================\u001b[36m http://subread.sourceforge.net/ \u001b[0m======================//\n",
      "\n",
      "SRR1055108\n",
      "Performing FastQC for SRR1055108\n",
      "Started analysis of SRR1055108_1.fastq.gz\n",
      "Approx 5% complete for SRR1055108_1.fastq.gz\n",
      "Approx 10% complete for SRR1055108_1.fastq.gz\n",
      "Approx 15% complete for SRR1055108_1.fastq.gz\n",
      "Approx 20% complete for SRR1055108_1.fastq.gz\n",
      "Approx 25% complete for SRR1055108_1.fastq.gz\n",
      "Approx 30% complete for SRR1055108_1.fastq.gz\n",
      "Approx 35% complete for SRR1055108_1.fastq.gz\n",
      "Approx 40% complete for SRR1055108_1.fastq.gz\n",
      "Approx 45% complete for SRR1055108_1.fastq.gz\n",
      "Approx 50% complete for SRR1055108_1.fastq.gz\n",
      "Approx 55% complete for SRR1055108_1.fastq.gz\n",
      "Approx 60% complete for SRR1055108_1.fastq.gz\n",
      "Approx 65% complete for SRR1055108_1.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 70% complete for SRR1055108_1.fastq.gz\n",
      "Approx 75% complete for SRR1055108_1.fastq.gz\n",
      "Approx 80% complete for SRR1055108_1.fastq.gz\n",
      "Approx 85% complete for SRR1055108_1.fastq.gz\n",
      "Approx 90% complete for SRR1055108_1.fastq.gz\n",
      "Approx 95% complete for SRR1055108_1.fastq.gz\n",
      "Analysis complete for SRR1055108_1.fastq.gz\n",
      "Started analysis of SRR1055108_2.fastq.gz\n",
      "Approx 5% complete for SRR1055108_2.fastq.gz\n",
      "Approx 10% complete for SRR1055108_2.fastq.gz\n",
      "Approx 15% complete for SRR1055108_2.fastq.gz\n",
      "Approx 20% complete for SRR1055108_2.fastq.gz\n",
      "Approx 25% complete for SRR1055108_2.fastq.gz\n",
      "Approx 30% complete for SRR1055108_2.fastq.gz\n",
      "Approx 35% complete for SRR1055108_2.fastq.gz\n",
      "Approx 40% complete for SRR1055108_2.fastq.gz\n",
      "Approx 45% complete for SRR1055108_2.fastq.gz\n",
      "Approx 50% complete for SRR1055108_2.fastq.gz\n",
      "Approx 55% complete for SRR1055108_2.fastq.gz\n",
      "Approx 60% complete for SRR1055108_2.fastq.gz\n",
      "Approx 65% complete for SRR1055108_2.fastq.gz\n",
      "Approx 70% complete for SRR1055108_2.fastq.gz\n",
      "Approx 75% complete for SRR1055108_2.fastq.gz\n",
      "Approx 80% complete for SRR1055108_2.fastq.gz\n",
      "Approx 85% complete for SRR1055108_2.fastq.gz\n",
      "Approx 90% complete for SRR1055108_2.fastq.gz\n",
      "Approx 95% complete for SRR1055108_2.fastq.gz\n",
      "Analysis complete for SRR1055108_2.fastq.gz\n",
      "Aligning reads from SRR1055108 to the reference genome\n",
      "Mar 18 00:34:14 ..... started STAR run\n",
      "Mar 18 00:34:14 ..... loading genome\n",
      "Mar 18 00:36:59 ..... processing annotations GTF\n",
      "Mar 18 00:37:00 ..... started mapping\n",
      "Mar 18 01:04:18 ..... finished successfully\n",
      "\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m    _____ _    _ ____  _____  ______          _____  \n",
      "       \u001b[44;37m =====      \u001b[0m\u001b[36m   / ____| |  | |  _ \\|  __ \\|  ____|   /\\   |  __ \\ \n",
      "       \u001b[44;37m   =====    \u001b[0m\u001b[36m  | (___ | |  | | |_) | |__) | |__     /  \\  | |  | |\n",
      "       \u001b[44;37m     ====   \u001b[0m\u001b[36m   \\___ \\| |  | |  _ <|  _  /|  __|   / /\\ \\ | |  | |\n",
      "       \u001b[44;37m       ==== \u001b[0m\u001b[36m   ____) | |__| | |_) | | \\ \\| |____ / ____ \\| |__| |\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m  |_____/ \\____/|____/|_|  \\_\\______/_/    \\_\\_____/\u001b[0m\n",
      "\t  v1.6.3\n",
      "\n",
      "//==========================\u001b[36m featureCounts setting \u001b[0m===========================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Input files : \u001b[36m1 BAM file  \u001b[0m \u001b[0m                                    ||\n",
      "||                           \u001b[32mP\u001b[36m SRR1055108Aligned.out.bam\u001b[0m \u001b[0m                     ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Output file : \u001b[36mSRR1055108.count.txt\u001b[0m \u001b[0m                            ||\n",
      "||                 Summary : \u001b[36mSRR1055108.count.txt.summary\u001b[0m \u001b[0m                    ||\n",
      "||              Annotation : \u001b[36msczGene.gtf (GTF)\u001b[0m \u001b[0m                               ||\n",
      "||      Dir for temp files : \u001b[36m../featureCount_output\u001b[0m \u001b[0m                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||                 Threads : \u001b[36m16\u001b[0m \u001b[0m                                              ||\n",
      "||                   Level : \u001b[36mmeta-feature level\u001b[0m \u001b[0m                              ||\n",
      "||              Paired-end : \u001b[36mno\u001b[0m \u001b[0m                                              ||\n",
      "||      Multimapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "|| Multi-overlapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "||   Min overlapping bases : \u001b[36m1\u001b[0m \u001b[0m                                               ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\=====================\u001b[36m http://subread.sourceforge.net/ \u001b[0m======================//\n",
      "\n",
      "//=================================\u001b[36m Running \u001b[0m==================================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Load annotation file sczGene.gtf \u001b[0m... \u001b[0m                                      ||\n",
      "||    Features : \u001b[36m33932\u001b[0m \u001b[0m                                                       ||\n",
      "||    Meta-features : \u001b[36m33932\u001b[0m \u001b[0m                                                  ||\n",
      "||    Chromosomes/contigs : \u001b[36m3116\u001b[0m \u001b[0m                                             ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Process BAM file SRR1055108Aligned.out.bam... \u001b[0m                             ||\n",
      "||    Paired-end reads are included. \u001b[0m                                         ||\n",
      "||    Assign alignments to features... \u001b[0m                                       ||\n",
      "||    Total alignments : \u001b[36m37722670\u001b[0m \u001b[0m                                            ||\n",
      "||    Successfully assigned alignments : \u001b[36m18970976 (50.3%)\u001b[0m \u001b[0m                    ||\n",
      "||    Running time : \u001b[36m0.40 minutes\u001b[0m \u001b[0m                                            ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Summary of counting results can be found in file \"../featureCount_output/ \u001b[0m ||\n",
      "|| SRR1055108.count.txt.summary\" \u001b[0m                                             ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\=====================\u001b[36m http://subread.sourceforge.net/ \u001b[0m======================//\n",
      "\n",
      "SRR1055109\n",
      "Performing FastQC for SRR1055109\n",
      "Started analysis of SRR1055109_1.fastq.gz\n",
      "Approx 5% complete for SRR1055109_1.fastq.gz\n",
      "Approx 10% complete for SRR1055109_1.fastq.gz\n",
      "Approx 15% complete for SRR1055109_1.fastq.gz\n",
      "Approx 20% complete for SRR1055109_1.fastq.gz\n",
      "Approx 25% complete for SRR1055109_1.fastq.gz\n",
      "Approx 30% complete for SRR1055109_1.fastq.gz\n",
      "Approx 35% complete for SRR1055109_1.fastq.gz\n",
      "Approx 40% complete for SRR1055109_1.fastq.gz\n",
      "Approx 45% complete for SRR1055109_1.fastq.gz\n",
      "Approx 50% complete for SRR1055109_1.fastq.gz\n",
      "Approx 55% complete for SRR1055109_1.fastq.gz\n",
      "Approx 60% complete for SRR1055109_1.fastq.gz\n",
      "Approx 65% complete for SRR1055109_1.fastq.gz\n",
      "Approx 70% complete for SRR1055109_1.fastq.gz\n",
      "Approx 75% complete for SRR1055109_1.fastq.gz\n",
      "Approx 80% complete for SRR1055109_1.fastq.gz\n",
      "Approx 85% complete for SRR1055109_1.fastq.gz\n",
      "Approx 90% complete for SRR1055109_1.fastq.gz\n",
      "Approx 95% complete for SRR1055109_1.fastq.gz\n",
      "Analysis complete for SRR1055109_1.fastq.gz\n",
      "Started analysis of SRR1055109_2.fastq.gz\n",
      "Approx 5% complete for SRR1055109_2.fastq.gz\n",
      "Approx 10% complete for SRR1055109_2.fastq.gz\n",
      "Approx 15% complete for SRR1055109_2.fastq.gz\n",
      "Approx 20% complete for SRR1055109_2.fastq.gz\n",
      "Approx 25% complete for SRR1055109_2.fastq.gz\n",
      "Approx 30% complete for SRR1055109_2.fastq.gz\n",
      "Approx 35% complete for SRR1055109_2.fastq.gz\n",
      "Approx 40% complete for SRR1055109_2.fastq.gz\n",
      "Approx 45% complete for SRR1055109_2.fastq.gz\n",
      "Approx 50% complete for SRR1055109_2.fastq.gz\n",
      "Approx 55% complete for SRR1055109_2.fastq.gz\n",
      "Approx 60% complete for SRR1055109_2.fastq.gz\n",
      "Approx 65% complete for SRR1055109_2.fastq.gz\n",
      "Approx 70% complete for SRR1055109_2.fastq.gz\n",
      "Approx 75% complete for SRR1055109_2.fastq.gz\n",
      "Approx 80% complete for SRR1055109_2.fastq.gz\n",
      "Approx 85% complete for SRR1055109_2.fastq.gz\n",
      "Approx 90% complete for SRR1055109_2.fastq.gz\n",
      "Approx 95% complete for SRR1055109_2.fastq.gz\n",
      "Analysis complete for SRR1055109_2.fastq.gz\n",
      "Aligning reads from SRR1055109 to the reference genome\n",
      "Mar 18 01:14:02 ..... started STAR run\n",
      "Mar 18 01:14:02 ..... loading genome\n",
      "Mar 18 01:16:45 ..... processing annotations GTF\n",
      "Mar 18 01:16:46 ..... started mapping\n",
      "Mar 18 01:50:17 ..... finished successfully\n",
      "\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m    _____ _    _ ____  _____  ______          _____  \n",
      "       \u001b[44;37m =====      \u001b[0m\u001b[36m   / ____| |  | |  _ \\|  __ \\|  ____|   /\\   |  __ \\ \n",
      "       \u001b[44;37m   =====    \u001b[0m\u001b[36m  | (___ | |  | | |_) | |__) | |__     /  \\  | |  | |\n",
      "       \u001b[44;37m     ====   \u001b[0m\u001b[36m   \\___ \\| |  | |  _ <|  _  /|  __|   / /\\ \\ | |  | |\n",
      "       \u001b[44;37m       ==== \u001b[0m\u001b[36m   ____) | |__| | |_) | | \\ \\| |____ / ____ \\| |__| |\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m  |_____/ \\____/|____/|_|  \\_\\______/_/    \\_\\_____/\u001b[0m\n",
      "\t  v1.6.3\n",
      "\n",
      "//==========================\u001b[36m featureCounts setting \u001b[0m===========================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Input files : \u001b[36m1 BAM file  \u001b[0m \u001b[0m                                    ||\n",
      "||                           \u001b[32mP\u001b[36m SRR1055109Aligned.out.bam\u001b[0m \u001b[0m                     ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Output file : \u001b[36mSRR1055109.count.txt\u001b[0m \u001b[0m                            ||\n",
      "||                 Summary : \u001b[36mSRR1055109.count.txt.summary\u001b[0m \u001b[0m                    ||\n",
      "||              Annotation : \u001b[36msczGene.gtf (GTF)\u001b[0m \u001b[0m                               ||\n",
      "||      Dir for temp files : \u001b[36m../featureCount_output\u001b[0m \u001b[0m                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||                 Threads : \u001b[36m16\u001b[0m \u001b[0m                                              ||\n",
      "||                   Level : \u001b[36mmeta-feature level\u001b[0m \u001b[0m                              ||\n",
      "||              Paired-end : \u001b[36mno\u001b[0m \u001b[0m                                              ||\n",
      "||      Multimapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "|| Multi-overlapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "||   Min overlapping bases : \u001b[36m1\u001b[0m \u001b[0m                                               ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\=====================\u001b[36m http://subread.sourceforge.net/ \u001b[0m======================//\n",
      "\n",
      "//=================================\u001b[36m Running \u001b[0m==================================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Load annotation file sczGene.gtf \u001b[0m... \u001b[0m                                      ||\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "||    Features : \u001b[36m33932\u001b[0m \u001b[0m                                                       ||\n",
      "||    Meta-features : \u001b[36m33932\u001b[0m \u001b[0m                                                  ||\n",
      "||    Chromosomes/contigs : \u001b[36m3116\u001b[0m \u001b[0m                                             ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Process BAM file SRR1055109Aligned.out.bam... \u001b[0m                             ||\n",
      "||    Paired-end reads are included. \u001b[0m                                         ||\n",
      "||    Assign alignments to features... \u001b[0m                                       ||\n",
      "||    Total alignments : \u001b[36m46631214\u001b[0m \u001b[0m                                            ||\n",
      "||    Successfully assigned alignments : \u001b[36m17297429 (37.1%)\u001b[0m \u001b[0m                    ||\n",
      "||    Running time : \u001b[36m0.68 minutes\u001b[0m \u001b[0m                                            ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Summary of counting results can be found in file \"../featureCount_output/ \u001b[0m ||\n",
      "|| SRR1055109.count.txt.summary\" \u001b[0m                                             ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\=====================\u001b[36m http://subread.sourceforge.net/ \u001b[0m======================//\n",
      "\n",
      "SRR1055110\n",
      "Performing FastQC for SRR1055110\n",
      "Started analysis of SRR1055110_1.fastq.gz\n",
      "Approx 5% complete for SRR1055110_1.fastq.gz\n",
      "Approx 10% complete for SRR1055110_1.fastq.gz\n",
      "Approx 15% complete for SRR1055110_1.fastq.gz\n",
      "Approx 20% complete for SRR1055110_1.fastq.gz\n",
      "Approx 25% complete for SRR1055110_1.fastq.gz\n",
      "Approx 30% complete for SRR1055110_1.fastq.gz\n",
      "Approx 35% complete for SRR1055110_1.fastq.gz\n",
      "Approx 40% complete for SRR1055110_1.fastq.gz\n",
      "Approx 45% complete for SRR1055110_1.fastq.gz\n",
      "Approx 50% complete for SRR1055110_1.fastq.gz\n",
      "Approx 55% complete for SRR1055110_1.fastq.gz\n",
      "Approx 60% complete for SRR1055110_1.fastq.gz\n",
      "Approx 65% complete for SRR1055110_1.fastq.gz\n",
      "Approx 70% complete for SRR1055110_1.fastq.gz\n",
      "Approx 75% complete for SRR1055110_1.fastq.gz\n",
      "Approx 80% complete for SRR1055110_1.fastq.gz\n",
      "Approx 85% complete for SRR1055110_1.fastq.gz\n",
      "Approx 90% complete for SRR1055110_1.fastq.gz\n",
      "Approx 95% complete for SRR1055110_1.fastq.gz\n",
      "Analysis complete for SRR1055110_1.fastq.gz\n",
      "Started analysis of SRR1055110_2.fastq.gz\n",
      "Approx 5% complete for SRR1055110_2.fastq.gz\n",
      "Approx 10% complete for SRR1055110_2.fastq.gz\n",
      "Approx 15% complete for SRR1055110_2.fastq.gz\n",
      "Approx 20% complete for SRR1055110_2.fastq.gz\n",
      "Approx 25% complete for SRR1055110_2.fastq.gz\n",
      "Approx 30% complete for SRR1055110_2.fastq.gz\n",
      "Approx 35% complete for SRR1055110_2.fastq.gz\n",
      "Approx 40% complete for SRR1055110_2.fastq.gz\n",
      "Approx 45% complete for SRR1055110_2.fastq.gz\n",
      "Approx 50% complete for SRR1055110_2.fastq.gz\n",
      "Approx 55% complete for SRR1055110_2.fastq.gz\n",
      "Approx 60% complete for SRR1055110_2.fastq.gz\n",
      "Approx 65% complete for SRR1055110_2.fastq.gz\n",
      "Approx 70% complete for SRR1055110_2.fastq.gz\n",
      "Approx 75% complete for SRR1055110_2.fastq.gz\n",
      "Approx 80% complete for SRR1055110_2.fastq.gz\n",
      "Approx 85% complete for SRR1055110_2.fastq.gz\n",
      "Approx 90% complete for SRR1055110_2.fastq.gz\n",
      "Approx 95% complete for SRR1055110_2.fastq.gz\n",
      "Analysis complete for SRR1055110_2.fastq.gz\n",
      "Aligning reads from SRR1055110 to the reference genome\n",
      "Mar 18 01:56:41 ..... started STAR run\n",
      "Mar 18 01:56:41 ..... loading genome\n",
      "Mar 18 01:59:25 ..... processing annotations GTF\n",
      "Mar 18 01:59:26 ..... started mapping\n",
      "Mar 18 02:16:37 ..... finished successfully\n",
      "\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m    _____ _    _ ____  _____  ______          _____  \n",
      "       \u001b[44;37m =====      \u001b[0m\u001b[36m   / ____| |  | |  _ \\|  __ \\|  ____|   /\\   |  __ \\ \n",
      "       \u001b[44;37m   =====    \u001b[0m\u001b[36m  | (___ | |  | | |_) | |__) | |__     /  \\  | |  | |\n",
      "       \u001b[44;37m     ====   \u001b[0m\u001b[36m   \\___ \\| |  | |  _ <|  _  /|  __|   / /\\ \\ | |  | |\n",
      "       \u001b[44;37m       ==== \u001b[0m\u001b[36m   ____) | |__| | |_) | | \\ \\| |____ / ____ \\| |__| |\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m  |_____/ \\____/|____/|_|  \\_\\______/_/    \\_\\_____/\u001b[0m\n",
      "\t  v1.6.3\n",
      "\n",
      "//==========================\u001b[36m featureCounts setting \u001b[0m===========================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Input files : \u001b[36m1 BAM file  \u001b[0m \u001b[0m                                    ||\n",
      "||                           \u001b[32mP\u001b[36m SRR1055110Aligned.out.bam\u001b[0m \u001b[0m                     ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Output file : \u001b[36mSRR1055110.count.txt\u001b[0m \u001b[0m                            ||\n",
      "||                 Summary : \u001b[36mSRR1055110.count.txt.summary\u001b[0m \u001b[0m                    ||\n",
      "||              Annotation : \u001b[36msczGene.gtf (GTF)\u001b[0m \u001b[0m                               ||\n",
      "||      Dir for temp files : \u001b[36m../featureCount_output\u001b[0m \u001b[0m                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||                 Threads : \u001b[36m16\u001b[0m \u001b[0m                                              ||\n",
      "||                   Level : \u001b[36mmeta-feature level\u001b[0m \u001b[0m                              ||\n",
      "||              Paired-end : \u001b[36mno\u001b[0m \u001b[0m                                              ||\n",
      "||      Multimapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "|| Multi-overlapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "||   Min overlapping bases : \u001b[36m1\u001b[0m \u001b[0m                                               ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\=====================\u001b[36m http://subread.sourceforge.net/ \u001b[0m======================//\n",
      "\n",
      "//=================================\u001b[36m Running \u001b[0m==================================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Load annotation file sczGene.gtf \u001b[0m... \u001b[0m                                      ||\n",
      "||    Features : \u001b[36m33932\u001b[0m \u001b[0m                                                       ||\n",
      "||    Meta-features : \u001b[36m33932\u001b[0m \u001b[0m                                                  ||\n",
      "||    Chromosomes/contigs : \u001b[36m3116\u001b[0m \u001b[0m                                             ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Process BAM file SRR1055110Aligned.out.bam... \u001b[0m                             ||\n",
      "||    Paired-end reads are included. \u001b[0m                                         ||\n",
      "||    Assign alignments to features... \u001b[0m                                       ||\n",
      "||    Total alignments : \u001b[36m23274552\u001b[0m \u001b[0m                                            ||\n",
      "||    Successfully assigned alignments : \u001b[36m14690684 (63.1%)\u001b[0m \u001b[0m                    ||\n",
      "||    Running time : \u001b[36m0.19 minutes\u001b[0m \u001b[0m                                            ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Summary of counting results can be found in file \"../featureCount_output/ \u001b[0m ||\n",
      "|| SRR1055110.count.txt.summary\" \u001b[0m                                             ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\=====================\u001b[36m http://subread.sourceforge.net/ \u001b[0m======================//\n",
      "\n",
      "SRR1055182\n",
      "Performing FastQC for SRR1055182\n",
      "Started analysis of SRR1055182_1.fastq.gz\n",
      "Approx 5% complete for SRR1055182_1.fastq.gz\n",
      "Approx 10% complete for SRR1055182_1.fastq.gz\n",
      "Approx 15% complete for SRR1055182_1.fastq.gz\n",
      "Approx 20% complete for SRR1055182_1.fastq.gz\n",
      "Approx 25% complete for SRR1055182_1.fastq.gz\n",
      "Approx 30% complete for SRR1055182_1.fastq.gz\n",
      "Approx 35% complete for SRR1055182_1.fastq.gz\n",
      "Approx 40% complete for SRR1055182_1.fastq.gz\n",
      "Approx 45% complete for SRR1055182_1.fastq.gz\n",
      "Approx 50% complete for SRR1055182_1.fastq.gz\n",
      "Approx 55% complete for SRR1055182_1.fastq.gz\n",
      "Approx 60% complete for SRR1055182_1.fastq.gz\n",
      "Approx 65% complete for SRR1055182_1.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 70% complete for SRR1055182_1.fastq.gz\n",
      "Approx 75% complete for SRR1055182_1.fastq.gz\n",
      "Approx 80% complete for SRR1055182_1.fastq.gz\n",
      "Approx 85% complete for SRR1055182_1.fastq.gz\n",
      "Approx 90% complete for SRR1055182_1.fastq.gz\n",
      "Approx 95% complete for SRR1055182_1.fastq.gz\n",
      "Analysis complete for SRR1055182_1.fastq.gz\n",
      "Started analysis of SRR1055182_2.fastq.gz\n",
      "Approx 5% complete for SRR1055182_2.fastq.gz\n",
      "Approx 10% complete for SRR1055182_2.fastq.gz\n",
      "Approx 15% complete for SRR1055182_2.fastq.gz\n",
      "Approx 20% complete for SRR1055182_2.fastq.gz\n",
      "Approx 25% complete for SRR1055182_2.fastq.gz\n",
      "Approx 30% complete for SRR1055182_2.fastq.gz\n",
      "Approx 35% complete for SRR1055182_2.fastq.gz\n",
      "Approx 40% complete for SRR1055182_2.fastq.gz\n",
      "Approx 45% complete for SRR1055182_2.fastq.gz\n",
      "Approx 50% complete for SRR1055182_2.fastq.gz\n",
      "Approx 55% complete for SRR1055182_2.fastq.gz\n",
      "Approx 60% complete for SRR1055182_2.fastq.gz\n",
      "Approx 65% complete for SRR1055182_2.fastq.gz\n",
      "Approx 70% complete for SRR1055182_2.fastq.gz\n",
      "Approx 75% complete for SRR1055182_2.fastq.gz\n",
      "Approx 80% complete for SRR1055182_2.fastq.gz\n",
      "Approx 85% complete for SRR1055182_2.fastq.gz\n",
      "Approx 90% complete for SRR1055182_2.fastq.gz\n",
      "Approx 95% complete for SRR1055182_2.fastq.gz\n",
      "Analysis complete for SRR1055182_2.fastq.gz\n",
      "Aligning reads from SRR1055182 to the reference genome\n",
      "Mar 18 02:26:14 ..... started STAR run\n",
      "Mar 18 02:26:14 ..... loading genome\n",
      "Mar 18 02:28:53 ..... processing annotations GTF\n",
      "Mar 18 02:28:54 ..... started mapping\n",
      "Mar 18 03:01:23 ..... finished successfully\n",
      "\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m    _____ _    _ ____  _____  ______          _____  \n",
      "       \u001b[44;37m =====      \u001b[0m\u001b[36m   / ____| |  | |  _ \\|  __ \\|  ____|   /\\   |  __ \\ \n",
      "       \u001b[44;37m   =====    \u001b[0m\u001b[36m  | (___ | |  | | |_) | |__) | |__     /  \\  | |  | |\n",
      "       \u001b[44;37m     ====   \u001b[0m\u001b[36m   \\___ \\| |  | |  _ <|  _  /|  __|   / /\\ \\ | |  | |\n",
      "       \u001b[44;37m       ==== \u001b[0m\u001b[36m   ____) | |__| | |_) | | \\ \\| |____ / ____ \\| |__| |\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m  |_____/ \\____/|____/|_|  \\_\\______/_/    \\_\\_____/\u001b[0m\n",
      "\t  v1.6.3\n",
      "\n",
      "//==========================\u001b[36m featureCounts setting \u001b[0m===========================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Input files : \u001b[36m1 BAM file  \u001b[0m \u001b[0m                                    ||\n",
      "||                           \u001b[32mP\u001b[36m SRR1055182Aligned.out.bam\u001b[0m \u001b[0m                     ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Output file : \u001b[36mSRR1055182.count.txt\u001b[0m \u001b[0m                            ||\n",
      "||                 Summary : \u001b[36mSRR1055182.count.txt.summary\u001b[0m \u001b[0m                    ||\n",
      "||              Annotation : \u001b[36msczGene.gtf (GTF)\u001b[0m \u001b[0m                               ||\n",
      "||      Dir for temp files : \u001b[36m../featureCount_output\u001b[0m \u001b[0m                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||                 Threads : \u001b[36m16\u001b[0m \u001b[0m                                              ||\n",
      "||                   Level : \u001b[36mmeta-feature level\u001b[0m \u001b[0m                              ||\n",
      "||              Paired-end : \u001b[36mno\u001b[0m \u001b[0m                                              ||\n",
      "||      Multimapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "|| Multi-overlapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "||   Min overlapping bases : \u001b[36m1\u001b[0m \u001b[0m                                               ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\=====================\u001b[36m http://subread.sourceforge.net/ \u001b[0m======================//\n",
      "\n",
      "//=================================\u001b[36m Running \u001b[0m==================================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Load annotation file sczGene.gtf \u001b[0m... \u001b[0m                                      ||\n",
      "||    Features : \u001b[36m33932\u001b[0m \u001b[0m                                                       ||\n",
      "||    Meta-features : \u001b[36m33932\u001b[0m \u001b[0m                                                  ||\n",
      "||    Chromosomes/contigs : \u001b[36m3116\u001b[0m \u001b[0m                                             ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Process BAM file SRR1055182Aligned.out.bam... \u001b[0m                             ||\n",
      "||    Paired-end reads are included. \u001b[0m                                         ||\n",
      "||    Assign alignments to features... \u001b[0m                                       ||\n",
      "||    Total alignments : \u001b[36m42049988\u001b[0m \u001b[0m                                            ||\n",
      "||    Successfully assigned alignments : \u001b[36m17777889 (42.3%)\u001b[0m \u001b[0m                    ||\n",
      "||    Running time : \u001b[36m0.68 minutes\u001b[0m \u001b[0m                                            ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Summary of counting results can be found in file \"../featureCount_output/ \u001b[0m ||\n",
      "|| SRR1055182.count.txt.summary\" \u001b[0m                                             ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\=====================\u001b[36m http://subread.sourceforge.net/ \u001b[0m======================//\n",
      "\n",
      "SRR1055932\n",
      "Performing FastQC for SRR1055932\n",
      "Started analysis of SRR1055932_1.fastq.gz\n",
      "Approx 5% complete for SRR1055932_1.fastq.gz\n",
      "Approx 10% complete for SRR1055932_1.fastq.gz\n",
      "Approx 15% complete for SRR1055932_1.fastq.gz\n",
      "Approx 20% complete for SRR1055932_1.fastq.gz\n",
      "Approx 25% complete for SRR1055932_1.fastq.gz\n",
      "Approx 30% complete for SRR1055932_1.fastq.gz\n",
      "Approx 35% complete for SRR1055932_1.fastq.gz\n",
      "Approx 40% complete for SRR1055932_1.fastq.gz\n",
      "Approx 45% complete for SRR1055932_1.fastq.gz\n",
      "Approx 50% complete for SRR1055932_1.fastq.gz\n",
      "Approx 55% complete for SRR1055932_1.fastq.gz\n",
      "Approx 60% complete for SRR1055932_1.fastq.gz\n",
      "Approx 65% complete for SRR1055932_1.fastq.gz\n",
      "Approx 70% complete for SRR1055932_1.fastq.gz\n",
      "Approx 75% complete for SRR1055932_1.fastq.gz\n",
      "Approx 80% complete for SRR1055932_1.fastq.gz\n",
      "Approx 85% complete for SRR1055932_1.fastq.gz\n",
      "Approx 90% complete for SRR1055932_1.fastq.gz\n",
      "Approx 95% complete for SRR1055932_1.fastq.gz\n",
      "Analysis complete for SRR1055932_1.fastq.gz\n",
      "Started analysis of SRR1055932_2.fastq.gz\n",
      "Approx 5% complete for SRR1055932_2.fastq.gz\n",
      "Approx 10% complete for SRR1055932_2.fastq.gz\n",
      "Approx 15% complete for SRR1055932_2.fastq.gz\n",
      "Approx 20% complete for SRR1055932_2.fastq.gz\n",
      "Approx 25% complete for SRR1055932_2.fastq.gz\n",
      "Approx 30% complete for SRR1055932_2.fastq.gz\n",
      "Approx 35% complete for SRR1055932_2.fastq.gz\n",
      "Approx 40% complete for SRR1055932_2.fastq.gz\n",
      "Approx 45% complete for SRR1055932_2.fastq.gz\n",
      "Approx 50% complete for SRR1055932_2.fastq.gz\n",
      "Approx 55% complete for SRR1055932_2.fastq.gz\n",
      "Approx 60% complete for SRR1055932_2.fastq.gz\n",
      "Approx 65% complete for SRR1055932_2.fastq.gz\n",
      "Approx 70% complete for SRR1055932_2.fastq.gz\n",
      "Approx 75% complete for SRR1055932_2.fastq.gz\n",
      "Approx 80% complete for SRR1055932_2.fastq.gz\n",
      "Approx 85% complete for SRR1055932_2.fastq.gz\n",
      "Approx 90% complete for SRR1055932_2.fastq.gz\n",
      "Approx 95% complete for SRR1055932_2.fastq.gz\n",
      "Analysis complete for SRR1055932_2.fastq.gz\n",
      "Aligning reads from SRR1055932 to the reference genome\n",
      "Mar 18 03:10:46 ..... started STAR run\n",
      "Mar 18 03:10:46 ..... loading genome\n",
      "Mar 18 03:13:28 ..... processing annotations GTF\n",
      "Mar 18 03:13:29 ..... started mapping\n",
      "Mar 18 03:47:11 ..... finished successfully\n",
      "\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m    _____ _    _ ____  _____  ______          _____  \n",
      "       \u001b[44;37m =====      \u001b[0m\u001b[36m   / ____| |  | |  _ \\|  __ \\|  ____|   /\\   |  __ \\ \n",
      "       \u001b[44;37m   =====    \u001b[0m\u001b[36m  | (___ | |  | | |_) | |__) | |__     /  \\  | |  | |\n",
      "       \u001b[44;37m     ====   \u001b[0m\u001b[36m   \\___ \\| |  | |  _ <|  _  /|  __|   / /\\ \\ | |  | |\n",
      "       \u001b[44;37m       ==== \u001b[0m\u001b[36m   ____) | |__| | |_) | | \\ \\| |____ / ____ \\| |__| |\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m  |_____/ \\____/|____/|_|  \\_\\______/_/    \\_\\_____/\u001b[0m\n",
      "\t  v1.6.3\n",
      "\n",
      "//==========================\u001b[36m featureCounts setting \u001b[0m===========================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Input files : \u001b[36m1 BAM file  \u001b[0m \u001b[0m                                    ||\n",
      "||                           \u001b[32mP\u001b[36m SRR1055932Aligned.out.bam\u001b[0m \u001b[0m                     ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Output file : \u001b[36mSRR1055932.count.txt\u001b[0m \u001b[0m                            ||\n",
      "||                 Summary : \u001b[36mSRR1055932.count.txt.summary\u001b[0m \u001b[0m                    ||\n",
      "||              Annotation : \u001b[36msczGene.gtf (GTF)\u001b[0m \u001b[0m                               ||\n",
      "||      Dir for temp files : \u001b[36m../featureCount_output\u001b[0m \u001b[0m                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||                 Threads : \u001b[36m16\u001b[0m \u001b[0m                                              ||\n",
      "||                   Level : \u001b[36mmeta-feature level\u001b[0m \u001b[0m                              ||\n",
      "||              Paired-end : \u001b[36mno\u001b[0m \u001b[0m                                              ||\n",
      "||      Multimapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "|| Multi-overlapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "||   Min overlapping bases : \u001b[36m1\u001b[0m \u001b[0m                                               ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\=====================\u001b[36m http://subread.sourceforge.net/ \u001b[0m======================//\n",
      "\n",
      "//=================================\u001b[36m Running \u001b[0m==================================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Load annotation file sczGene.gtf \u001b[0m... \u001b[0m                                      ||\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "||    Features : \u001b[36m33932\u001b[0m \u001b[0m                                                       ||\n",
      "||    Meta-features : \u001b[36m33932\u001b[0m \u001b[0m                                                  ||\n",
      "||    Chromosomes/contigs : \u001b[36m3116\u001b[0m \u001b[0m                                             ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Process BAM file SRR1055932Aligned.out.bam... \u001b[0m                             ||\n",
      "||    Paired-end reads are included. \u001b[0m                                         ||\n",
      "||    Assign alignments to features... \u001b[0m                                       ||\n",
      "||    Total alignments : \u001b[36m42910954\u001b[0m \u001b[0m                                            ||\n",
      "||    Successfully assigned alignments : \u001b[36m17572537 (41.0%)\u001b[0m \u001b[0m                    ||\n",
      "||    Running time : \u001b[36m0.62 minutes\u001b[0m \u001b[0m                                            ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Summary of counting results can be found in file \"../featureCount_output/ \u001b[0m ||\n",
      "|| SRR1055932.count.txt.summary\" \u001b[0m                                             ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\=====================\u001b[36m http://subread.sourceforge.net/ \u001b[0m======================//\n",
      "\n",
      "SRR1055933\n",
      "Performing FastQC for SRR1055933\n",
      "Started analysis of SRR1055933_1.fastq.gz\n",
      "Approx 5% complete for SRR1055933_1.fastq.gz\n",
      "Approx 10% complete for SRR1055933_1.fastq.gz\n",
      "Approx 15% complete for SRR1055933_1.fastq.gz\n",
      "Approx 20% complete for SRR1055933_1.fastq.gz\n",
      "Approx 25% complete for SRR1055933_1.fastq.gz\n",
      "Approx 30% complete for SRR1055933_1.fastq.gz\n",
      "Approx 35% complete for SRR1055933_1.fastq.gz\n",
      "Approx 40% complete for SRR1055933_1.fastq.gz\n",
      "Approx 45% complete for SRR1055933_1.fastq.gz\n",
      "Approx 50% complete for SRR1055933_1.fastq.gz\n",
      "Approx 55% complete for SRR1055933_1.fastq.gz\n",
      "Approx 60% complete for SRR1055933_1.fastq.gz\n",
      "Approx 65% complete for SRR1055933_1.fastq.gz\n",
      "Approx 70% complete for SRR1055933_1.fastq.gz\n",
      "Approx 75% complete for SRR1055933_1.fastq.gz\n",
      "Approx 80% complete for SRR1055933_1.fastq.gz\n",
      "Approx 85% complete for SRR1055933_1.fastq.gz\n",
      "Approx 90% complete for SRR1055933_1.fastq.gz\n",
      "Approx 95% complete for SRR1055933_1.fastq.gz\n",
      "Analysis complete for SRR1055933_1.fastq.gz\n",
      "Started analysis of SRR1055933_2.fastq.gz\n",
      "Approx 5% complete for SRR1055933_2.fastq.gz\n",
      "Approx 10% complete for SRR1055933_2.fastq.gz\n",
      "Approx 15% complete for SRR1055933_2.fastq.gz\n",
      "Approx 20% complete for SRR1055933_2.fastq.gz\n",
      "Approx 25% complete for SRR1055933_2.fastq.gz\n",
      "Approx 30% complete for SRR1055933_2.fastq.gz\n",
      "Approx 35% complete for SRR1055933_2.fastq.gz\n",
      "Approx 40% complete for SRR1055933_2.fastq.gz\n",
      "Approx 45% complete for SRR1055933_2.fastq.gz\n",
      "Approx 50% complete for SRR1055933_2.fastq.gz\n",
      "Approx 55% complete for SRR1055933_2.fastq.gz\n",
      "Approx 60% complete for SRR1055933_2.fastq.gz\n",
      "Approx 65% complete for SRR1055933_2.fastq.gz\n",
      "Approx 70% complete for SRR1055933_2.fastq.gz\n",
      "Approx 75% complete for SRR1055933_2.fastq.gz\n",
      "Approx 80% complete for SRR1055933_2.fastq.gz\n",
      "Approx 85% complete for SRR1055933_2.fastq.gz\n",
      "Approx 90% complete for SRR1055933_2.fastq.gz\n",
      "Approx 95% complete for SRR1055933_2.fastq.gz\n",
      "Analysis complete for SRR1055933_2.fastq.gz\n",
      "Aligning reads from SRR1055933 to the reference genome\n",
      "Mar 18 03:54:00 ..... started STAR run\n",
      "Mar 18 03:54:00 ..... loading genome\n",
      "Mar 18 03:56:44 ..... processing annotations GTF\n",
      "Mar 18 03:56:45 ..... started mapping\n",
      "Mar 18 04:14:11 ..... finished successfully\n",
      "\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m    _____ _    _ ____  _____  ______          _____  \n",
      "       \u001b[44;37m =====      \u001b[0m\u001b[36m   / ____| |  | |  _ \\|  __ \\|  ____|   /\\   |  __ \\ \n",
      "       \u001b[44;37m   =====    \u001b[0m\u001b[36m  | (___ | |  | | |_) | |__) | |__     /  \\  | |  | |\n",
      "       \u001b[44;37m     ====   \u001b[0m\u001b[36m   \\___ \\| |  | |  _ <|  _  /|  __|   / /\\ \\ | |  | |\n",
      "       \u001b[44;37m       ==== \u001b[0m\u001b[36m   ____) | |__| | |_) | | \\ \\| |____ / ____ \\| |__| |\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m  |_____/ \\____/|____/|_|  \\_\\______/_/    \\_\\_____/\u001b[0m\n",
      "\t  v1.6.3\n",
      "\n",
      "//==========================\u001b[36m featureCounts setting \u001b[0m===========================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Input files : \u001b[36m1 BAM file  \u001b[0m \u001b[0m                                    ||\n",
      "||                           \u001b[32mP\u001b[36m SRR1055933Aligned.out.bam\u001b[0m \u001b[0m                     ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Output file : \u001b[36mSRR1055933.count.txt\u001b[0m \u001b[0m                            ||\n",
      "||                 Summary : \u001b[36mSRR1055933.count.txt.summary\u001b[0m \u001b[0m                    ||\n",
      "||              Annotation : \u001b[36msczGene.gtf (GTF)\u001b[0m \u001b[0m                               ||\n",
      "||      Dir for temp files : \u001b[36m../featureCount_output\u001b[0m \u001b[0m                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||                 Threads : \u001b[36m16\u001b[0m \u001b[0m                                              ||\n",
      "||                   Level : \u001b[36mmeta-feature level\u001b[0m \u001b[0m                              ||\n",
      "||              Paired-end : \u001b[36mno\u001b[0m \u001b[0m                                              ||\n",
      "||      Multimapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "|| Multi-overlapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "||   Min overlapping bases : \u001b[36m1\u001b[0m \u001b[0m                                               ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\=====================\u001b[36m http://subread.sourceforge.net/ \u001b[0m======================//\n",
      "\n",
      "//=================================\u001b[36m Running \u001b[0m==================================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Load annotation file sczGene.gtf \u001b[0m... \u001b[0m                                      ||\n",
      "||    Features : \u001b[36m33932\u001b[0m \u001b[0m                                                       ||\n",
      "||    Meta-features : \u001b[36m33932\u001b[0m \u001b[0m                                                  ||\n",
      "||    Chromosomes/contigs : \u001b[36m3116\u001b[0m \u001b[0m                                             ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Process BAM file SRR1055933Aligned.out.bam... \u001b[0m                             ||\n",
      "||    Paired-end reads are included. \u001b[0m                                         ||\n",
      "||    Assign alignments to features... \u001b[0m                                       ||\n",
      "||    Total alignments : \u001b[36m21625010\u001b[0m \u001b[0m                                            ||\n",
      "||    Successfully assigned alignments : \u001b[36m14389234 (66.5%)\u001b[0m \u001b[0m                    ||\n",
      "||    Running time : \u001b[36m0.18 minutes\u001b[0m \u001b[0m                                            ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Summary of counting results can be found in file \"../featureCount_output/ \u001b[0m ||\n",
      "|| SRR1055933.count.txt.summary\" \u001b[0m                                             ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\=====================\u001b[36m http://subread.sourceforge.net/ \u001b[0m======================//\n",
      "\n",
      "SRR1055934\n",
      "Performing FastQC for SRR1055934\n",
      "Started analysis of SRR1055934_1.fastq.gz\n",
      "Approx 5% complete for SRR1055934_1.fastq.gz\n",
      "Approx 10% complete for SRR1055934_1.fastq.gz\n",
      "Approx 15% complete for SRR1055934_1.fastq.gz\n",
      "Approx 20% complete for SRR1055934_1.fastq.gz\n",
      "Approx 25% complete for SRR1055934_1.fastq.gz\n",
      "Approx 30% complete for SRR1055934_1.fastq.gz\n",
      "Approx 35% complete for SRR1055934_1.fastq.gz\n",
      "Approx 40% complete for SRR1055934_1.fastq.gz\n",
      "Approx 45% complete for SRR1055934_1.fastq.gz\n",
      "Approx 50% complete for SRR1055934_1.fastq.gz\n",
      "Approx 55% complete for SRR1055934_1.fastq.gz\n",
      "Approx 60% complete for SRR1055934_1.fastq.gz\n",
      "Approx 65% complete for SRR1055934_1.fastq.gz\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Approx 70% complete for SRR1055934_1.fastq.gz\n",
      "Approx 75% complete for SRR1055934_1.fastq.gz\n",
      "Approx 80% complete for SRR1055934_1.fastq.gz\n",
      "Approx 85% complete for SRR1055934_1.fastq.gz\n",
      "Approx 90% complete for SRR1055934_1.fastq.gz\n",
      "Approx 95% complete for SRR1055934_1.fastq.gz\n",
      "Analysis complete for SRR1055934_1.fastq.gz\n",
      "Started analysis of SRR1055934_2.fastq.gz\n",
      "Approx 5% complete for SRR1055934_2.fastq.gz\n",
      "Approx 10% complete for SRR1055934_2.fastq.gz\n",
      "Approx 15% complete for SRR1055934_2.fastq.gz\n",
      "Approx 20% complete for SRR1055934_2.fastq.gz\n",
      "Approx 25% complete for SRR1055934_2.fastq.gz\n",
      "Approx 30% complete for SRR1055934_2.fastq.gz\n",
      "Approx 35% complete for SRR1055934_2.fastq.gz\n",
      "Approx 40% complete for SRR1055934_2.fastq.gz\n",
      "Approx 45% complete for SRR1055934_2.fastq.gz\n",
      "Approx 50% complete for SRR1055934_2.fastq.gz\n",
      "Approx 55% complete for SRR1055934_2.fastq.gz\n",
      "Approx 60% complete for SRR1055934_2.fastq.gz\n",
      "Approx 65% complete for SRR1055934_2.fastq.gz\n",
      "Approx 70% complete for SRR1055934_2.fastq.gz\n",
      "Approx 75% complete for SRR1055934_2.fastq.gz\n",
      "Approx 80% complete for SRR1055934_2.fastq.gz\n",
      "Approx 85% complete for SRR1055934_2.fastq.gz\n",
      "Approx 90% complete for SRR1055934_2.fastq.gz\n",
      "Approx 95% complete for SRR1055934_2.fastq.gz\n",
      "Analysis complete for SRR1055934_2.fastq.gz\n",
      "Aligning reads from SRR1055934 to the reference genome\n",
      "Mar 18 04:23:06 ..... started STAR run\n",
      "Mar 18 04:23:06 ..... loading genome\n",
      "Mar 18 04:26:06 ..... processing annotations GTF\n",
      "Mar 18 04:26:08 ..... started mapping\n",
      "Mar 18 04:47:47 ..... finished successfully\n",
      "\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m    _____ _    _ ____  _____  ______          _____  \n",
      "       \u001b[44;37m =====      \u001b[0m\u001b[36m   / ____| |  | |  _ \\|  __ \\|  ____|   /\\   |  __ \\ \n",
      "       \u001b[44;37m   =====    \u001b[0m\u001b[36m  | (___ | |  | | |_) | |__) | |__     /  \\  | |  | |\n",
      "       \u001b[44;37m     ====   \u001b[0m\u001b[36m   \\___ \\| |  | |  _ <|  _  /|  __|   / /\\ \\ | |  | |\n",
      "       \u001b[44;37m       ==== \u001b[0m\u001b[36m   ____) | |__| | |_) | | \\ \\| |____ / ____ \\| |__| |\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m  |_____/ \\____/|____/|_|  \\_\\______/_/    \\_\\_____/\u001b[0m\n",
      "\t  v1.6.3\n",
      "\n",
      "//==========================\u001b[36m featureCounts setting \u001b[0m===========================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Input files : \u001b[36m1 BAM file  \u001b[0m \u001b[0m                                    ||\n",
      "||                           \u001b[32mP\u001b[36m SRR1055934Aligned.out.bam\u001b[0m \u001b[0m                     ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Output file : \u001b[36mSRR1055934.count.txt\u001b[0m \u001b[0m                            ||\n",
      "||                 Summary : \u001b[36mSRR1055934.count.txt.summary\u001b[0m \u001b[0m                    ||\n",
      "||              Annotation : \u001b[36msczGene.gtf (GTF)\u001b[0m \u001b[0m                               ||\n",
      "||      Dir for temp files : \u001b[36m../featureCount_output\u001b[0m \u001b[0m                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||                 Threads : \u001b[36m16\u001b[0m \u001b[0m                                              ||\n",
      "||                   Level : \u001b[36mmeta-feature level\u001b[0m \u001b[0m                              ||\n",
      "||              Paired-end : \u001b[36mno\u001b[0m \u001b[0m                                              ||\n",
      "||      Multimapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "|| Multi-overlapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "||   Min overlapping bases : \u001b[36m1\u001b[0m \u001b[0m                                               ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\=====================\u001b[36m http://subread.sourceforge.net/ \u001b[0m======================//\n",
      "\n",
      "//=================================\u001b[36m Running \u001b[0m==================================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Load annotation file sczGene.gtf \u001b[0m... \u001b[0m                                      ||\n",
      "||    Features : \u001b[36m33932\u001b[0m \u001b[0m                                                       ||\n",
      "||    Meta-features : \u001b[36m33932\u001b[0m \u001b[0m                                                  ||\n",
      "||    Chromosomes/contigs : \u001b[36m3116\u001b[0m \u001b[0m                                             ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Process BAM file SRR1055934Aligned.out.bam... \u001b[0m                             ||\n",
      "||    Paired-end reads are included. \u001b[0m                                         ||\n",
      "||    Assign alignments to features... \u001b[0m                                       ||\n",
      "||    Total alignments : \u001b[36m30666920\u001b[0m \u001b[0m                                            ||\n",
      "||    Successfully assigned alignments : \u001b[36m20825928 (67.9%)\u001b[0m \u001b[0m                    ||\n",
      "||    Running time : \u001b[36m0.46 minutes\u001b[0m \u001b[0m                                            ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Summary of counting results can be found in file \"../featureCount_output/ \u001b[0m ||\n",
      "|| SRR1055934.count.txt.summary\" \u001b[0m                                             ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\=====================\u001b[36m http://subread.sourceforge.net/ \u001b[0m======================//\n",
      "\n",
      "SRR1055944\n",
      "Performing FastQC for SRR1055944\n",
      "Started analysis of SRR1055944_1.fastq.gz\n",
      "Approx 5% complete for SRR1055944_1.fastq.gz\n",
      "Approx 10% complete for SRR1055944_1.fastq.gz\n",
      "Approx 15% complete for SRR1055944_1.fastq.gz\n",
      "Approx 20% complete for SRR1055944_1.fastq.gz\n",
      "Approx 25% complete for SRR1055944_1.fastq.gz\n",
      "Approx 30% complete for SRR1055944_1.fastq.gz\n",
      "Approx 35% complete for SRR1055944_1.fastq.gz\n",
      "Approx 40% complete for SRR1055944_1.fastq.gz\n",
      "Approx 45% complete for SRR1055944_1.fastq.gz\n",
      "Approx 50% complete for SRR1055944_1.fastq.gz\n",
      "Approx 55% complete for SRR1055944_1.fastq.gz\n",
      "Approx 60% complete for SRR1055944_1.fastq.gz\n",
      "Approx 65% complete for SRR1055944_1.fastq.gz\n",
      "Approx 70% complete for SRR1055944_1.fastq.gz\n",
      "Approx 75% complete for SRR1055944_1.fastq.gz\n",
      "Approx 80% complete for SRR1055944_1.fastq.gz\n",
      "Approx 85% complete for SRR1055944_1.fastq.gz\n",
      "Approx 90% complete for SRR1055944_1.fastq.gz\n",
      "Approx 95% complete for SRR1055944_1.fastq.gz\n",
      "Analysis complete for SRR1055944_1.fastq.gz\n",
      "Started analysis of SRR1055944_2.fastq.gz\n",
      "Approx 5% complete for SRR1055944_2.fastq.gz\n",
      "Approx 10% complete for SRR1055944_2.fastq.gz\n",
      "Approx 15% complete for SRR1055944_2.fastq.gz\n",
      "Approx 20% complete for SRR1055944_2.fastq.gz\n",
      "Approx 25% complete for SRR1055944_2.fastq.gz\n",
      "Approx 30% complete for SRR1055944_2.fastq.gz\n",
      "Approx 35% complete for SRR1055944_2.fastq.gz\n",
      "Approx 40% complete for SRR1055944_2.fastq.gz\n",
      "Approx 45% complete for SRR1055944_2.fastq.gz\n",
      "Approx 50% complete for SRR1055944_2.fastq.gz\n",
      "Approx 55% complete for SRR1055944_2.fastq.gz\n",
      "Approx 60% complete for SRR1055944_2.fastq.gz\n",
      "Approx 65% complete for SRR1055944_2.fastq.gz\n",
      "Approx 70% complete for SRR1055944_2.fastq.gz\n",
      "Approx 75% complete for SRR1055944_2.fastq.gz\n",
      "Approx 80% complete for SRR1055944_2.fastq.gz\n",
      "Approx 85% complete for SRR1055944_2.fastq.gz\n",
      "Approx 90% complete for SRR1055944_2.fastq.gz\n",
      "Approx 95% complete for SRR1055944_2.fastq.gz\n",
      "Analysis complete for SRR1055944_2.fastq.gz\n",
      "Aligning reads from SRR1055944 to the reference genome\n",
      "Mar 18 04:59:37 ..... started STAR run\n",
      "Mar 18 04:59:38 ..... loading genome\n",
      "Mar 18 05:02:30 ..... processing annotations GTF\n",
      "Mar 18 05:02:31 ..... started mapping\n",
      "Mar 18 05:46:48 ..... finished successfully\n",
      "\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m    _____ _    _ ____  _____  ______          _____  \n",
      "       \u001b[44;37m =====      \u001b[0m\u001b[36m   / ____| |  | |  _ \\|  __ \\|  ____|   /\\   |  __ \\ \n",
      "       \u001b[44;37m   =====    \u001b[0m\u001b[36m  | (___ | |  | | |_) | |__) | |__     /  \\  | |  | |\n",
      "       \u001b[44;37m     ====   \u001b[0m\u001b[36m   \\___ \\| |  | |  _ <|  _  /|  __|   / /\\ \\ | |  | |\n",
      "       \u001b[44;37m       ==== \u001b[0m\u001b[36m   ____) | |__| | |_) | | \\ \\| |____ / ____ \\| |__| |\n",
      "       \u001b[44;37m ========== \u001b[0m\u001b[36m  |_____/ \\____/|____/|_|  \\_\\______/_/    \\_\\_____/\u001b[0m\n",
      "\t  v1.6.3\n",
      "\n",
      "//==========================\u001b[36m featureCounts setting \u001b[0m===========================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Input files : \u001b[36m1 BAM file  \u001b[0m \u001b[0m                                    ||\n",
      "||                           \u001b[32mP\u001b[36m SRR1055944Aligned.out.bam\u001b[0m \u001b[0m                     ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||             Output file : \u001b[36mSRR1055944.count.txt\u001b[0m \u001b[0m                            ||\n",
      "||                 Summary : \u001b[36mSRR1055944.count.txt.summary\u001b[0m \u001b[0m                    ||\n",
      "||              Annotation : \u001b[36msczGene.gtf (GTF)\u001b[0m \u001b[0m                               ||\n",
      "||      Dir for temp files : \u001b[36m../featureCount_output\u001b[0m \u001b[0m                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||                 Threads : \u001b[36m16\u001b[0m \u001b[0m                                              ||\n",
      "||                   Level : \u001b[36mmeta-feature level\u001b[0m \u001b[0m                              ||\n",
      "||              Paired-end : \u001b[36mno\u001b[0m \u001b[0m                                              ||\n",
      "||      Multimapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "|| Multi-overlapping reads : \u001b[36mnot counted\u001b[0m \u001b[0m                                     ||\n",
      "||   Min overlapping bases : \u001b[36m1\u001b[0m \u001b[0m                                               ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\=====================\u001b[36m http://subread.sourceforge.net/ \u001b[0m======================//\n",
      "\n",
      "//=================================\u001b[36m Running \u001b[0m==================================\\\\\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Load annotation file sczGene.gtf \u001b[0m... \u001b[0m                                      ||\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "||    Features : \u001b[36m33932\u001b[0m \u001b[0m                                                       ||\n",
      "||    Meta-features : \u001b[36m33932\u001b[0m \u001b[0m                                                  ||\n",
      "||    Chromosomes/contigs : \u001b[36m3116\u001b[0m \u001b[0m                                             ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Process BAM file SRR1055944Aligned.out.bam... \u001b[0m                             ||\n",
      "||    Paired-end reads are included. \u001b[0m                                         ||\n",
      "||    Assign alignments to features... \u001b[0m                                       ||\n",
      "||    Total alignments : \u001b[36m58784102\u001b[0m \u001b[0m                                            ||\n",
      "||    Successfully assigned alignments : \u001b[36m19502902 (33.2%)\u001b[0m \u001b[0m                    ||\n",
      "||    Running time : \u001b[36m1.08 minutes\u001b[0m \u001b[0m                                            ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "|| Summary of counting results can be found in file \"../featureCount_output/ \u001b[0m ||\n",
      "|| SRR1055944.count.txt.summary\" \u001b[0m                                             ||\n",
      "||  \u001b[0m                                                                          ||\n",
      "\\\\=====================\u001b[36m http://subread.sourceforge.net/ \u001b[0m======================//\n",
      "\n"
     ]
    }
   ],
   "source": [
    "! bash analyze_sra.sh -t 16  | tee analyze_sra.log"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "data/fastQC_output/<br>\n",
       "&nbsp;&nbsp;<a href='data/fastQC_output/SRR1051214_1_fastqc.html' target='_blank'>SRR1051214_1_fastqc.html</a><br>\n",
       "&nbsp;&nbsp;<a href='data/fastQC_output/SRR1051214_2_fastqc.html' target='_blank'>SRR1051214_2_fastqc.html</a><br>\n",
       "&nbsp;&nbsp;<a href='data/fastQC_output/SRR1053623_1_fastqc.html' target='_blank'>SRR1053623_1_fastqc.html</a><br>\n",
       "&nbsp;&nbsp;<a href='data/fastQC_output/SRR1053623_2_fastqc.html' target='_blank'>SRR1053623_2_fastqc.html</a><br>\n",
       "&nbsp;&nbsp;<a href='data/fastQC_output/SRR1054007_1_fastqc.html' target='_blank'>SRR1054007_1_fastqc.html</a><br>\n",
       "&nbsp;&nbsp;<a href='data/fastQC_output/SRR1054007_2_fastqc.html' target='_blank'>SRR1054007_2_fastqc.html</a><br>\n",
       "&nbsp;&nbsp;<a href='data/fastQC_output/SRR1054086_1_fastqc.html' target='_blank'>SRR1054086_1_fastqc.html</a><br>\n",
       "&nbsp;&nbsp;<a href='data/fastQC_output/SRR1054086_2_fastqc.html' target='_blank'>SRR1054086_2_fastqc.html</a><br>\n",
       "&nbsp;&nbsp;<a href='data/fastQC_output/SRR1054152_1_fastqc.html' target='_blank'>SRR1054152_1_fastqc.html</a><br>\n",
       "&nbsp;&nbsp;<a href='data/fastQC_output/SRR1054152_2_fastqc.html' target='_blank'>SRR1054152_2_fastqc.html</a><br>\n",
       "&nbsp;&nbsp;<a href='data/fastQC_output/SRR1055108_1_fastqc.html' target='_blank'>SRR1055108_1_fastqc.html</a><br>\n",
       "&nbsp;&nbsp;<a href='data/fastQC_output/SRR1055108_2_fastqc.html' target='_blank'>SRR1055108_2_fastqc.html</a><br>\n",
       "&nbsp;&nbsp;<a href='data/fastQC_output/SRR1055109_1_fastqc.html' target='_blank'>SRR1055109_1_fastqc.html</a><br>\n",
       "&nbsp;&nbsp;<a href='data/fastQC_output/SRR1055109_2_fastqc.html' target='_blank'>SRR1055109_2_fastqc.html</a><br>\n",
       "&nbsp;&nbsp;<a href='data/fastQC_output/SRR1055110_1_fastqc.html' target='_blank'>SRR1055110_1_fastqc.html</a><br>\n",
       "&nbsp;&nbsp;<a href='data/fastQC_output/SRR1055110_2_fastqc.html' target='_blank'>SRR1055110_2_fastqc.html</a><br>\n",
       "&nbsp;&nbsp;<a href='data/fastQC_output/SRR1055182_1_fastqc.html' target='_blank'>SRR1055182_1_fastqc.html</a><br>\n",
       "&nbsp;&nbsp;<a href='data/fastQC_output/SRR1055182_2_fastqc.html' target='_blank'>SRR1055182_2_fastqc.html</a><br>\n",
       "&nbsp;&nbsp;<a href='data/fastQC_output/SRR1055932_1_fastqc.html' target='_blank'>SRR1055932_1_fastqc.html</a><br>\n",
       "&nbsp;&nbsp;<a href='data/fastQC_output/SRR1055932_2_fastqc.html' target='_blank'>SRR1055932_2_fastqc.html</a><br>\n",
       "&nbsp;&nbsp;<a href='data/fastQC_output/SRR1055933_1_fastqc.html' target='_blank'>SRR1055933_1_fastqc.html</a><br>\n",
       "&nbsp;&nbsp;<a href='data/fastQC_output/SRR1055933_2_fastqc.html' target='_blank'>SRR1055933_2_fastqc.html</a><br>\n",
       "&nbsp;&nbsp;<a href='data/fastQC_output/SRR1055934_1_fastqc.html' target='_blank'>SRR1055934_1_fastqc.html</a><br>\n",
       "&nbsp;&nbsp;<a href='data/fastQC_output/SRR1055934_2_fastqc.html' target='_blank'>SRR1055934_2_fastqc.html</a><br>\n",
       "&nbsp;&nbsp;<a href='data/fastQC_output/SRR1055944_1_fastqc.html' target='_blank'>SRR1055944_1_fastqc.html</a><br>\n",
       "&nbsp;&nbsp;<a href='data/fastQC_output/SRR1055944_2_fastqc.html' target='_blank'>SRR1055944_2_fastqc.html</a><br>"
      ],
      "text/plain": [
       "data/fastQC_output/\n",
       "  SRR1051214_1_fastqc.html\n",
       "  SRR1051214_2_fastqc.html\n",
       "  SRR1053623_1_fastqc.html\n",
       "  SRR1053623_2_fastqc.html\n",
       "  SRR1054007_1_fastqc.html\n",
       "  SRR1054007_2_fastqc.html\n",
       "  SRR1054086_1_fastqc.html\n",
       "  SRR1054086_2_fastqc.html\n",
       "  SRR1054152_1_fastqc.html\n",
       "  SRR1054152_2_fastqc.html\n",
       "  SRR1055108_1_fastqc.html\n",
       "  SRR1055108_2_fastqc.html\n",
       "  SRR1055109_1_fastqc.html\n",
       "  SRR1055109_2_fastqc.html\n",
       "  SRR1055110_1_fastqc.html\n",
       "  SRR1055110_2_fastqc.html\n",
       "  SRR1055182_1_fastqc.html\n",
       "  SRR1055182_2_fastqc.html\n",
       "  SRR1055932_1_fastqc.html\n",
       "  SRR1055932_2_fastqc.html\n",
       "  SRR1055933_1_fastqc.html\n",
       "  SRR1055933_2_fastqc.html\n",
       "  SRR1055934_1_fastqc.html\n",
       "  SRR1055934_2_fastqc.html\n",
       "  SRR1055944_1_fastqc.html\n",
       "  SRR1055944_2_fastqc.html"
      ]
     },
     "execution_count": 3,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Examining the QC reports to evaluate the quality of the data:\n",
    "import os\n",
    "import numpy as np\n",
    "import pandas as pd\n",
    "\n",
    "os.environ['WORKDIR'] = \"data/\"\n",
    "os.environ['GENOMEDIR'] = \"$HOME/genomes/\"\n",
    "\n",
    "from IPython.display import FileLinks\n",
    "FileLinks(os.path.join(os.environ['WORKDIR'], 'fastQC_output'), included_suffixes=['.html'])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Setting WORKDIR to: /home/meixin/gadProperty/data/ \n",
      "载入需要的程辑包：limma\n",
      "Finished reading featureCount into data.frame with shape:  33932 14 \n"
     ]
    }
   ],
   "source": [
    "# Merge counts:\n",
    "! Rscript merge.R"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Search GAD id:\n",
    "! grep 'glutamate decarboxylase' C.sinensis.gene.kegg.annotation.xls | cut -f1 | sort | uniq"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Normalization and Extract GAD counts:\n",
    "! Rscript gadExpr.R"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.0"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
